# ![ibiology -](https://www.ibiology.org/wp-content/uploads/2023/08/iBiology-Horizontal-Logo-B.png) Microscopy Series
> *Here you can find my note from the courses on [ibiology](https://www.ibiology.org) - [Microscopy Series](https://www.ibiology.org/online-biology-courses/microscopy-series/microscopy-series-table-contents/).*
>
> **Other useful links:**
> - [ComponentLibrary (a vector graphics library for illustrations of optics experiments)](http://www.gwoptics.org/ComponentLibrary/)
> - [Demystifying Microscopes: Disassembling a Nikon Ti Eclipse Microscope](https://www.ibiology.org/talks/nikon-ti-eclipse-microscope/) | [YouTube](https://youtu.be/DAV0X1c3s-g?si=pcsGO-8EVUfC9A5e)
> - [Physics of Nondestructive Evaluation](https://www.nde-ed.org/Physics/index.xhtml)
> - [openstax.org - Introduction to Geometric Optics](https://openstax.org/books/college-physics-2e/pages/25-introduction-to-geometric-optics)
> - [lumenlearning.com - Physics](https://courses.lumenlearning.com/suny-physics/)
> - [Ansys Education Resources](https://www.ansys.com/academic/educators/education-resources#q=optics&t=EducationResourcesTab&sort=relevancy&layout=card&numberOfResults=15)
> - [MicroscopyU - the source for microscopy education | Nikon](https://www.microscopyu.com/microscopy-basics)
> - [Miniaturized two-photon microscope: seeing clearer and deeper into the brain](https://www.nature.com/articles/lsa2017104)
> - [Miniature Multiphoton Microscopes for Recording Neural Activity in Freely Moving Animals](https://link.springer.com/protocol/10.1007/978-1-0716-2764-8_7)
> - [ Bioimage Analysis - Christian Tischer (EMBL)](https://www.youtube.com/watch?v=0PP38Z0CNMI)
>
> **Contents:**
> - [Historical Contributions from Light Microscopy: What Can You Learn with a Light Microscope? (Ron Vale, HHMI/UCSF)](https://github.com/gabboraron/ibiology---Microscopy-Series#historical-contributions-from-light-microscopy-what-can-you-learn-with-a-light-microscope-ron-vale-hhmiucsf)
> - [History of Early Microscopes (Joseph Gall, Carnegie Institution)](https://github.com/gabboraron/ibiology---Microscopy-Series#history-of-early-microscopes-joseph-gall-carnegie-institution)
> - [Refractive Lenses and Image Formation (Daniel Fletcher, UC Berkeley)](https://github.com/gabboraron/ibiology---Microscopy-Series#refractive-lenses-and-image-formation-daniel-fletcher-uc-berkeley)
> - [Microscope Imaging and Koehler Illumination (Ron Vale, HHMI/UCSF)](https://github.com/gabboraron/ibiology---Microscopy-Series#microscope-imaging-and-koehler-illumination-ron-vale-hhmiucsf)
> - [Objective Lenses and Eyepieces - Steve Ross](https://github.com/gabboraron/ibiology---Microscopy-Series#objective-lenses-and-eyepieces---steve-ross)
> - [Huygens Wavelets Constructive/Destructive Interference, and Diffraction](https://github.com/gabboraron/ibiology---Microscopy-Series#huygens-wavelets-constructivedestructive-interference-and-diffraction)
> - [Point Spread Function (Jeff Lichtman, Harvard University)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#point-spread-function-jeff-lichtman-harvard-university)
> - [Resolution of a Microscope (Jeff Lichtman, Harvard University)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#resolution-of-a-microscope-jeff-lichtman-harvard-university)
> - [Bonus: Photons: What is Light? (Bo Huang, UCSF)](https://github.com/gabboraron/ibiology---Microscopy-Series#bonus-photons-what-is-light-bo-huang-ucsf)
> - [Bonus: Fourier Space: Fourier Transform (Bo Huang, UCSF)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#bonus-fourier-space-fourier-transform-bo-huang-ucsf)
> - [Darkfield and Phase Contrast Microscopy (Edward Salmon, University of North Carolina)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#darkfield-and-phase-contrast-microscopy-edward-salmon-university-of-north-carolina)
> - [Polarized Light and its Interaction with Material (Shinya Inoue, Marine Biological Laboratory)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#polarized-light-and-its-interaction-with-material-shinya-inoue-marine-biological-laboratory)
> - [Polarization Microscopy (Edward Salmon, University of North Carolina)](https://github.com/gabboraron/ibiology---Microscopy-Series#polarization-microscopy-edward-salmon-university-of-north-carolina)
> - [Differential Interference Contrast (DIC) Microscopy (Edward Salmon, University of North Carolina)](https://github.com/gabboraron/ibiology---Microscopy-Series#differential-interference-contrast-dic-microscopy-edward-salmon-university-of-north-carolina)
> - [Bonus: Examples of Using Polarization Microscopy (Shinya Inoue, Marine Biological Laboratory)](https://github.com/gabboraron/ibiology---Microscopy-Series#differential-interference-contrast-dic-microscopy-edward-salmon-university-of-north-carolina)
> - [Introduction to Fluorescence Microscopy (Nico Stuurman, UCSF/HHMI)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#introduction-to-fluorescence-microscopy-nico-stuurman-ucsfhhmi)
> - [Fluorescent Probes: Organic Dyes and Quantum Dots (Timothy Mitchison, Harvard University)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main#fluorescent-probes-organic-dyes-and-quantum-dots-timothy-mitchison-harvard-university)
> - [Fluorescent Proteins and the Story Behind GFP (Roger Tsien, UCSD/HHMI)](https://github.com/gabboraron/ibiology---Microscopy-Series#fluorescent-proteins-and-the-story-behind-gfp-roger-tsien-ucsdhhmi)
> - [Optical Sectioning and Confocal Microscopy (Kurt Thorn, UCSF)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main?tab=readme-ov-file#optical-sectioning-and-confocal-microscopy-kurt-thorn-ucsf)
> - [Two Photon Microscopy (Kurt Thorn, UCSF)](https://github.com/gabboraron/ibiology---Microscopy-Series/tree/main?tab=readme-ov-file#two-photon-microscopy-kurt-thorn-ucsf)
>
> ---------

## Historical Contributions from Light Microscopy: What Can You Learn with a Light Microscope? (Ron Vale, HHMI/UCSF)
> [link](https://www.ibiology.org/talks/light-microscopy/) | [YouTube](https://www.youtube.com/watch?v=4c5ILWQmqRY&t=7s)
>
> *"You can observe a lot just by watching"*

Resolving power of the light microscope; the [resolution limits of light](https://en.wikipedia.org/wiki/Diffraction-limited_system): $d=\frac{\lambda}{2nsin\alpha}$ where $d$ is the distance about how far apart two objects are where they can still be distinguised as separate objects; $λ$ is the wavelength of light; $n$ is the index of refraction of the medium being imaged in; and $\alpha$ is the half-angle subtended by the optical objective lens. Here $nsin\alpha$ is the [numerical aperture](https://en.wikipedia.org/wiki/Numerical_aperture) that characterizes the range of angles over which the system can accept or emit light.

Green Fluorescence Protein is not compatible with the live cell imaging but is a subtatial improvement over immonuflourescence.

[Sliding filament theory of muscle contraction](https://www.youtube.com/watch?v=nTZnBdeIb5c)


## History of Early Microscopes (Joseph Gall, Carnegie Institution)
> [link](https://www.ibiology.org/talks/early-microscopes) | [YouTube](https://www.youtube.com/watch?v=U73iwy3YqCA)

| Late 1600s  - [Hooke's Microscope](https://micro.magnet.fsu.edu/primer/museum/hooke.html)| 1760 - [Screw-Barell microscope](https://www.microscopehistory.com/screw-barrel-1) | 1820 - [Brown's single lens microscope](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/Ford-B_Hidden-Realities-Brown-Micro_Pulse-iss-40-Dec-2018.pdf) and [discovery](https://physerver.hamilton.edu/research/brownian/Jigglyread.html) | 1838 - cell theory | 
| --------- | --------- | --------- | --------- | 
| ![Hook's microscope](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/hooke.jpg) | ![Screw-Barell microscope](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Screw-Barellmic.png) | ![Brown's microscope](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Brownmiclinn.png) | ![abbesthory](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/abbestheory.jpg) |
| Hook's ["Micrographia" book](https://www.gutenberg.org/ebooks/15491) is a set of 38 plates | more compaund microscopes arrives | | [Abbe’s Theory](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/Mart%C3%ADn_Albaladejo_C_Abbes-Theory.pdf) of image formation: A diffraction pattern is formed at the rear focal plane of the objective lens. The fine details of the image are observed when the diffracted rays from the specimen are recombined in the image. [*more*](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/pioneers_in_optics_ernst_abbe_18401905.pdf) | 

## Refractive Lenses and Image Formation (Daniel Fletcher, UC Berkeley)
> [lecture link](https://www.ibiology.org/talks/refractive-lenses/) | [YouTube](https://www.youtube.com/watch?v=vlxMSz7XRys&t=100s) | [questions](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/exercises/Refractive%20Lenses%20and%20Image%20Formation.jpg)
>
> Light microscopes use lenses. The basic properties of light, how light interacts with matter, the principles behind refractive lenses and how lenses form (magnified) images will be introduced in this talk.


> ***These equations will tell you where the light it's going to go.***

speed = wavelength (m) x frequency (1/s)

intensity = power / area

amplitude = $\sqrt{intensity}$

frequency = $\frac{speed}{wavelength}$

refraction *(sign:* $n$ *)* = $\frac{\text{speed of light in vacuum}}{\text{speed of light in material}}$

### Light - matter interaction:

**[law of reflection](https://www.britannica.com/science/reflection-physics): $\phi_{in} = \phi_{out}$**

The angle of incidence is equal with the reflection.

|![reflection_light_angle](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/reflection-light-angle-incidence.png)|![reflection in video](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/refractive_lenses_and_image_formation/light-matter_reflection.png)|
|--|--|


**Refraction is the bending of light (it also happens with sound, water and other waves) as it passes from one transparent substance into another. [Law of refrection](https://www.britannica.com/science/refraction): $n_isin(\phi_i) = n_rsin(\phi_t)$** In this [equation](https://farside.ph.utexas.edu/teaching/316/lectures/node128.html) $\phi_1$ is the angle subtended between the incident ray and the normal to the interface, and $\phi_2$ is the angle subtended between the refracted ray and the normal to the interface. The quantities $n_1$ and $n_2$ are termed the refractive indices of media 1 and 2, respectively. Thus, the law of refraction predicts that a light-ray always deviates more towards the normal in the optically denser medium: i.e., the medium with the higher refractive index. By definition, the refractive index $n$ of a dielectric medium of dielectric constant $K$ is given by $n=\sqrt{K}$. [Table](https://farside.ph.utexas.edu/teaching/316/lectures/node128.html#t12.1) below shows the refractive indices of some common materials (for yellow light of wavelength $\lambda =589$ nm).

| material | $n$ |
| --- | --- |
| air | 1.00029 |
| water | 1.33 |
| ice | 1.31 |
| glass | 1.52 |
| light flint | 1.58 |
| heavy flint | 1.65 |
| heaviest flint | 1.89 |
| diamond | 2.42 |

The law of refraction follows directly from the fact that the speed $v$ with which light propagates through a dielectric medium is inversely proportional to the refractive index of the medium $v = \frac{c}{n}$, where $c$ is the speed of the light.

| ![refraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Refraction-of-light-in-water20150805-30610-expmepedited.png) | ![low of refraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/law-refraction-angle-light-ray-Snell-medium.gif) |
|---- | ---- | 

[**The amount of bending depends on two things:**](https://www.sciencelearn.org.nz/resources/49-refraction-of-light)
- **Change in speed** – if a substance causes the light to speed up or slow down more, it will refract (bend) more.
- **Angle of the incident ray** – if the light is entering the substance at a greater angle, the amount of refraction will also be more noticeable. On the other hand, if the light is entering the new substance from straight on (at 90° to the surface), the light will still slow down, but it won’t change direction at all.

The law of refraction, also known as [Snell’s law](https://www.britannica.com/science/Snells-law), describes the relationship between the angle of incidence ($\phi_1$) and the angle of refraction ($\phi_2$), measured with respect to the normal (“perpendicular line”) to the surface, in mathematical terms: $n_1 sin \phi_1 = n_2 sin \phi_2$, where $n_1$ and $n_2$ are the index of refraction of the first and second media, respectively.

> ***These equations will tell you how much light will going to.***

$$transmitted + reflected + absorbed = \text{total amount of light}$$

$$\text{(reflection)  }R=(\frac{n_t-n_i}{n_t+n_i})^2$$

[Here](https://en.wikipedia.org/wiki/Fresnel_equations#Normal_incidence) assume the $\phi_i = \phi_t = 0$.

$$T=\frac{4n_i n_t}{(n_t+n_i)^2}$$

When the light goes throught several surfaces we can sum is as different refractions. This happens wen we use lenses and it is described by the [lensmaker's equation](http://www.physicsbootcamp.org/lens-maker-equation.html): $\frac{1}{f} = (n-1)(\frac{1}{R_1}-\frac{1}{R_2})$ where $f$ is the focal length, and $R_1$, $R_2$ are the radius of the circle which create the lens. When one surface of the lens is flat than then it means that you can set $R_2 = \infty$. Also you can set the thickness of the lens by $\frac{1}{f} = \frac{(n_1-n_0)}{n_0}(\frac{1}{R_1}-\frac{1}{R_2} + \frac{(n_1-n_0)d}{n_1R_1R_2})$  If the lens is operating in air, then set $n_0=1$, like in the previous equation.

| ![focal length - screenshot from video](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/refractive_lenses_and_image_formation/focal_length.png) | ![focal length](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/lens-maker-equation-geometry-for-focal-length.png) | 
| --- | --- |


At the focus point the light is has a distribution density.

![focus_point](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/refractive_lenses_and_image_formation/focus_point.png)

**Ray tracing:** parallel rays will go to the focus after lens and and will be parallel after lens if it comes from a focus.

*Thin lens equation:* $\frac{1}{f} = (\frac{1}{s_0} + \frac{1}{s_i})$ where $s_0$ is the distance from the object to the lens, and $s_i$ is the distance from lens to image and $f$ is the focal length of the lens.

*magnification:* $M=\frac{y_i}{y_0}=- \frac{s_i}{s_0}$ where $y_0$ is the object and $y_i$ is the image height.

The relationship between the focal lengths of the objective lens ($f_{obj}$), tube lens ($f_{tube}$), and the total magnification (M) is given by the formula: $M=-\frac{f_{tube}}{f_{obj}}$

![ray tracing](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/refractive_lenses_and_image_formation/ray_tracing.png)

## Microscope Imaging and Koehler Illumination (Ron Vale, HHMI/UCSF)
> [lecture link](https://www.ibiology.org/talks/microscope-imaging-koehler-illumination/) | [YouTube](https://www.youtube.com/watch?v=H5-CfX3XLf0)
>
> This lecture covers the lenses of the microscope and how they are used to focus light onto a specimen and how light from the specimen is focused onto a camera or the retina of an eye.  Koehler illumination, which provides even illumination of a sample, is described.


*info:* many microscope lens created earlier than 20 years has a 160mm image distance.

Nowdays mostly we use infinity lens whith tube lens. In this case the magnification will be $M=\frac{f_1}{f_0}$ where $f_1$ is the focal length of the tube lens and $f_0$ is the focal length of the objective lens which is a much smaller number. 

| ![tube lens](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscope%20Imaging%20and%20Koehler%20Illumination%20(Ron%20Vale)/tubelens.png) | ![projecting_the_imge_to_your_eye](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscope%20Imaging%20and%20Koehler%20Illumination%20(Ron%20Vale)/projecting_the_imge_to_your_eye.png) | ![projecting_the_imge_to_camera](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscope%20Imaging%20and%20Koehler%20Illumination%20(Ron%20Vale)/projecting_the_imge_to_camera.png) |
| ----- | ---- | ---- |

Here as you see we used [eyepiece](https://en.wikipedia.org/wiki/Eyepiece), we will talk about this later.


### The other question is the illumination of the microscope

With the Köhler illumination we can have defocused light  which don't disturb the image.

| ![köhler_illumination](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscope%20Imaging%20and%20Koehler%20Illumination%20(Ron%20Vale)/keohler.png) | ![köhler illumination, the path of the light and the path of the image](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/1024px-Kohler_Illumination_en.svg.png) |
| --------------------- | --------- | 
| [Köhler illumination](https://www.scientifica.uk.com/learning-zone/a-6-step-guide-to-koehler-illumination) | [Schematics of Köhler illumination.](https://en.wikipedia.org/wiki/K%C3%B6hler_illumination) Top: Illumination beam path, conjugated planes marked with the light green bar. Bottom: Imaging beam path, conjugated planes marked with the light blue bar. See text for further details. |

[Critical illumination](https://en.wikipedia.org/wiki/Critical_illumination) (below), focuses an image of a light source on to the specimen for bright illumination. Critical illumination generally has problems with evenness of illumination as an image of the illumination source (for example a halogen lamp filament) is visible in the resulting image. Köhler illumination has largely replaced critical illumination in modern scientific light microscopy although it requires additional optics which may not be present in less expensive and simpler light microscopes.

| ![critical_illumination.png](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscope%20Imaging%20and%20Koehler%20Illumination%20(Ron%20Vale)/critical_illumination.png) | ![critical illumination schematic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Critical_Illumination.svg) |
| ---- | ---- |

*info:* [Five steps for Koehler Illumination](https://www.leica-microsystems.com/science-lab/microscopy-basics/koehler-illumination-a-brief-history-and-a-practical-set-up-in-five-easy-steps/)

### Conjugate focal plane
> In optics, a [conjugate plane or conjugate focal plane](https://en.wikipedia.org/wiki/Conjugate_focal_plane) of a given plane P, is the plane P′ such that points on P are imaged on P′. If an object is moved to the point occupied by its image, then the moved object's new image will appear at the point where the object originated.

The field diaphragm simply controls the amount of light which finally reaches the slide on the stage. By controlling the amount of light coming from the light source, you are able to reduce glare from the source and this results in images with greater contrast. The field diaphragm is usually controlled by a knurled ring around the lens.

| ![iris_field](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscope%20Imaging%20and%20Koehler%20Illumination%20(Ron%20Vale)/iris_field.png) | ![The field diaphragm](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Koehler%20Illumination%20A%20Brief%20History%20and%20a%20Practical%20Set%20Up%20in%20Five%20Easy%20Steps.png) |
| ---- | ----- |
| The image of field diaphragm has an image on the sample and on the camer as well, but this works only with proper Köhler illumination. | [The main components of an upright microscope.](https://www.leica-microsystems.com/science-lab/microscopy-basics/koehler-illumination-a-brief-history-and-a-practical-set-up-in-five-easy-steps/) The Field diaphragm controls the intensity of light from the source. The condenser focusses the light from the light source on the specimen. The aperture diaphragm controls the angle of the illumination light cone.  |

## Objective Lenses and Eyepieces - Steve Ross
> [Lecture page](https://www.ibiology.org/talks/objective-lenses/) | [YouTube](https://www.youtube.com/watch?v=Y2tn7Prw1GA&embeds_referring_euri=https%3A%2F%2Fwww.ibiology.org%2F&source_ve_path=MjM4NTE&feature=emb_title)
>
> Microscope lenses are highly complex optical elements that correct for many different optical aberrations. This lecture describes a number of these aberrations and discusses what you should consider when selecting objective lenses as well as other precautions you can take to reduce the effects of aberrations.

### Optical Aberrations
It is always a tradeoff between the correction versus transmission and numerical aperture. Plan version are flat-field corrected, so that when you look in the eyepiece you can see a flat corrected field.

Apochromat lenses are the more correctted lenses, but don't forget, more glass in the lens to correct the lens, more transmission is losed!

| ![lenstypesandcorrections](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/lenstypesandcorrections.png) | ![aberrations](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/aberration.png) | ![numerical aperture](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/numericalaperture.png) |
| ------------ | ------------ | ------------------ |

#### Numerical aperture
> More in [ Encyclopedia of Materials: Science and Technology (Second Edition)](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/Optical_Microscopy-AS_HOLIK.pdf)
 
$$NA = nsin\phi$$
Where $NA$ is [numerical aperture](https://www.sciencedirect.com/sdfe/pdf/download/eid/3-s2.0-B0080431526011426/first-page-pdf), $n$ is the refractive index of the medium between the objective lens and the object *(air = 1.0; water = 1.33, oil = 1.515)*; $sin\phi$ is the angle of the cone of illumination; $\phi$ is half the angular aperture (or acceptance angle of image-forming rays) of the objective lens. This is why we can increase the magnification with immersion media.

| ![numerical_aperture](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/numerical_aperture.png) | ![immersion_media](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/immersion_media.png) |
| ------------------- |----- |

#### Curvature of field 
Due the curved surface of the lens you cant get the focus of the whole image at the same time.

Curvature of field is present when the sharpest image is formed not on a flat plane but on a curved surface. See in the thrd image, the image "plane" (the arc) deviates from a flat surface (the vertical line).

| ![fieldcurvature](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/fieldcurvature.png) | ![Curvature of field](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/aberration-curvature-field.gif) | ![Field curvature: the image "plane" (the arc) deviates from a flat surface (the vertical line).](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Field_curvature.svg) |
| ------------------- |----- | --- |

#### Astigmatism
A point source of the light which appears as a line or ellipse, and when you go through focus it changes its directionality in other way. High quality lenses don't heve such, just if you drop it. Occurs when thelens is curved more steeply in one direction than in another. [On the right picture](https://en.wikipedia.org/wiki/Astigmatism_(optical_systems)) you can see the image of point G is at point $B_M$ for rays in plane M, and at point $B_S$ for rays in plane S.

| ![astigmatism](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/astigmatism.png) | ![Astigmatism: The image of point G is at point BM for rays in plane M, and at point BS for rays in plane S.](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Meridional%2BSagittalPlane.png) |
| ------------------- |----- |

#### Coma
Coma of a single lens: each cone of light focuses on different planes along the optical axis.

| ![coma](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/coma.png) | ![coma](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Lens_coma.svg) |
| ------------------- |----- |

#### Axial chromatic and spherical aberration

| ![spherical aberration](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Spherical-aberration-disk.jpg) |A point source as imaged by a system with negative (top), zero (center), and positive (bottom) spherical aberration. Images to the left are defocused toward the inside, images on the right toward the outside|
| --- | --- |

Each manufacturer matching the immersion oil to their lenses, because other type can  induce axial chromatic aberration. The spherical aberration appears because in the reality there is no such thing as ideal thin lens, they have volume and curved surfaces. Common causes of spherical aberration  are the incorrect slip thickness, mismatch of the reflective index. 

| ![chromatic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/chromatic.png) | ![wrongimmersionoilinducechromaticaberration](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/influencesontheopticalpathlength.png) 
| ------------------- |----- |

| ![spherical](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/spherical.png) | ![spherical_aberration](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/spherical_aberration.png) | ![spherical_in_reality](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/spherical_in_reality.png) |
|----- |----- |----- |

Because of this nowdays the lens systems are extreamly complex systems. To solve this we have to know the optical path length. The $OPL$ is the physical length of every material in that optical system, times its refractive index, summed up through the entire lens system to the point in the specimen that you'are imaging. 

$$OPL = ((L_1 \times n_1) + (L_2 \times n_2) ... + (L_n \times n_n))$$

Where $L$ is the physical distance, $n$ is the refractive index of the mounting media *(air = 1.0; water = 1.33, glycerin=1.47, oil = 1.515)*
But in this equation the immersion media is a variable, immersion media decreases its refractive index as it warms up! Coverglasses ha a 10% $\pm$ thickness range. The specimen may vary in a long range, depends on the material.

| ![opl](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/opl.png) | ![influencesontheopticalpathlength](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/influencesontheopticalpathlength.png) | ![spherical_corrected](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Objectives%20and%20Eyepieces%20(Stephen%20Ross)/spherical_corrected.png) |
|----- |----- | ----- | 

## Huygens Wavelets Constructive/Destructive Interference, and Diffraction
> [lecture site](https://www.ibiology.org/talks/diffraction/) | [YouTube](https://www.youtube.com/watch?v=V8JXPqDWFcM) | [KhanAcademy](https://www.khanacademy.org/science/in-in-class-12th-physics-india/in-in-wave-optics) | [plane waves and spherical waves book](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/plane%20waves%20and%20spherica%20waves.pdf) | [openstax](https://openstax.org/books/college-physics-2e/pages/27-introduction-to-wave-optics) | [prof Sean Victor Hum - Spherical Waves](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/prof_Sean_Victor_Hum-spherical_waves.pdf) | [Dr. Daniel Mittleman - Reflection and Refraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/lecture12_1-Reflection%20and%20Refraction.pdf) 
>
> Light has properties of particles and waves. Understanding the wave nature of light is essential to understanding the workings of a microscope. This lecture describes Huygens Wavelets, constructive/destructive interference, and diffraction.

$$d \cdot sin \theta = n \lambda$$
[Where](https://isaacphysics.org/concepts/cp_diffraction?stage=all): $d$ is the distance between slits; $\theta$ is diffraction angle; $n$ is the order number for the maximum; $\lambda$ wavelength

A [pinhole camera, or camera obscura](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Pinhole-camera.svg%20-%20Wikipedia.png) simply create an image about the object if you put a paper in a good focus distance.

| ![light trough pinhole](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Light%20behaviour%20through%20pinhole.svg%20-%20Wikipedia.png) | ![pinhole camera](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Pinhole-camera.svg%20-%20Wikipedia.png) |
| --- | --- |

This is the good example for the light is not travelling in rays, because if it would then we can get a larger image if we move the paper backward. As we can see in the [wave optis](https://www.sciencedirect.com/topics/physics-and-astronomy/wave-optics), we have plane waves, because it is not just going up and down, but it is a 3D imensional thing, and there are a lot of planes behind each other.


![waveoptics](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/waveoptics.png)

[Plane waves](https://www.nde-ed.org/Physics/Waves/planewaves.xhtml) are a special case of waves where a physical quantity, such as phase, is constant over a plane that is perpendicular to the direction of wave travel. There can be mechanical (both transverseand longitudinal) as well as EM plane waves. It should be noted that EM plane waves are only transverse. A visualization of a plane wave is shown below. Just like periodic waves, plane waves have a wavelength, frequency, and wave speed. 

| ![planewave](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/planewave.png) | ![plane wave structure](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/plane%20wave.png) |
| ---- | --- | 

When EM waves radiate from a source, such as an antenna, they radiate as [spherical waves](https://academo.org/demos/spherical-waves/) (see the figure below). As the waves move farther from the source, their energy gets spread out over a bigger spherical surface area. At big distances, which we call the far-field, a spherical wave front can be approximated as a uniform plane wave over a defined area.  

| ![spherical vs plane wave](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/plane_waves.gif) | ![spherical wave](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/spherical%20wave.png) |
| ---- | --- | 

Here clearly we can see that how the waves "power" is missed if we increase the surface. [The equation below](https://www.soundonsound.com/sound-advice/q-does-strength-proximity-effect-depend-volume-source) describe this: $$intensity = \frac{P}{4\pi r^2}$$ where $P$ is the point source power.

> ### To understand better this, we have some basic definitions
>
> #### Velocity
> The distance covered by the wave in unit time.
>
> #### Frequency
> The number of complete disturbances (cycles) in unit time. Usually expressed in Hertz
>
> #### Wavelength
> The distance between successive points of equal phase in a wave.

$$\text{wavelenght } (\lambda) = \frac{\text{Velocity (v)}}{Frequency (f)}$$

### waves

Why intensity of a light wave is proportional to the $amplitude^2$ ? Generating a 4X larger wave requires 4 times the force over four times the distance = 16 times the energy. When such a wave imparts its energy to a detector we measure its enery as intensity: hence 4 times the amplitude is 16 times as brightness!

The intensity of a wave is the wavelet. The wavelet is the component that generates plane waves and spherical waves, [as Hyugens wrote it](https://www.nature.com/articles/s41598-021-99049-7). 

| ![hyugens](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/hyugens.png) | ![wavelet](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/wavelet.png) | ![hyugens wave](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Wavelength%3Dslitwidth.gif) | 
| ---- | --- |--- |

### Interference
The waves can make each other "stronger" or "weaker" based on the point where they intersect. This makes the [pinholes](https://en.wikipedia.org/wiki/Pinhole_(optics)) able to work. 
![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Waventerference.gif)

| ![pinhole](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/pinhole.png) | ![pinholes](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/pinholes.png) |
| ---- | ----- |
| ![pinhole_waves](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/pinhole_waves.png) | ![image_of_pinholes](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Huygens%20Wavelets%20ConstructiveDestructive%20Interference%2C%20and%20Diffraction/image_of_pinholes.png) |

## Point Spread Function (Jeff Lichtman, Harvard University)
> [course site](https://www.ibiology.org/talks/point-spread-function/) | [YouTube](https://www.youtube.com/watch?v=JQy94K94nL0) | [lumenlearning.com ~ Physics -  *Extended Topic* Microscopy Enhanced by the Wave Characteristics of Light](https://courses.lumenlearning.com/suny-physics/chapter/27-9-extended-topic-microscopy-enhanced-by-the-wave-characteristics-of-light/)
>
> An infinitesimally small point appears in the microscope as a spot with a certain size, blurred in the z-direction and with concentric rings around it. This “point spread function” reveals many of the optical properties of your microscope. This lecture explains why and how the microscope images a point as a point spread function.

| ![wave_optic_view_of_effect](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/wave_optic_view_of_effect.png) | ![huygens](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/huygens.png) | ![effect_two_wavelets](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/effect_two_wavelets.png) |
| ---------- | ---- |---- |
| Here the *Objective* is a microsope objective, which is a complex lens system basically. The *Objective* is exactly in the focal distance, from the *point source*, that's why we can say that the point is in focus for this lens. In this image, this lens turns the [spherical wave](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/lecture12_1-Reflection%20and%20Refraction.pdf) into [plane waves](https://en.wikipedia.org/wiki/Plane_wave). The plane waves arrive to a tube lens, which can take plane wave and turn into a converging spherical wave, so the tube lens focused on that point in the image plane. | What happens with the light beyond the tube lens: THe Huygen's view converging the spherical wave a series of infinitesimally small wavelets. Each of the infinite points in the emerging wave front acts like a spoint source itself. Each point emits a [wavelet](https://en.wikipedia.org/wiki/Wavelet), and all wavelets from the same wave front are mutually coherent. That's why they ascillate in synchrony, because they came from the same source. Therefore they interfere with each other in a predictable way we can quantify where the light will be and where the light won't be over a wavelength's time slot. | If there are two wavelets equally distant from the center line than those point both be peaking and troughing in synchrony and thei phase will be exactly the same. If we add it, they will add in the constructive interference. But if we choose a lower position then on of the waves will have a longer wavelength, in total they will have a half wavelength difference, creating a destructive interference. But again lower we can see that the wavelength is exactly one wavelength longer, so it will help each other creating a constructive interference. If we see this in every point then we can see that this happens in all of the points on the image plane. |

> ### Point spread function [(PSF)](https://en.wikipedia.org/wiki/Point_spread_function)
> 
> *Generally, a source of light can be considered a [point source](https://en.wikipedia.org/wiki/Point_source) if the resolution of the imaging instrument is too low to resolve the source's apparent size. There are two types and sources of light: a point source and an extended source. Mathematically an object may be considered a point source if its angular size,* $\theta$ , *is much smaller than the [resolving power of the telescope](https://unacademy.com/content/neet-ug/study-material/physics/resolving-power-of-microscopes-and-astronomical-telescopes/):* $$\theta << \frac{\lambda}{D}$$ *where* $\lambda$ *is the wavelength of light and* $D$ *is the diameter of telescope. Remember: telescopes having a larger* $D$ *or large diameter of the lens have a better resolving power. It also depends on light’s wavelength.*
>
> [Matematics of PSF](https://sci-hub.se/10.1109/TTHZ.2017.2750690)

| ![interference_from_two_wavelets](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/interference_from_two_wavelets.png) | ![numericalapertures](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/numericalapertures.png) | ![na_works](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/na_works.png) |
| ----- | ---- |---- |
| Down here you can see the integration of wavelets  overa full wavelength end then squered so we see amplitudes rather than wavelengths, which is a different pattern then the instantenous wave pattern, what we can see at the top. Detectors collect intensity. Intensity is a full cycle squered. The detected interference pattern has dark and bright lines. There is one single bright line running right down the horizontal middle, because that's equdistant from the two points everywhere along that line. The dark places are with destructive interference, where is no light, and the bright areas are therefore the constructive interference. | Those wavelets determinate the resolution of the microscope  are most related to the numerical aperture of the lens. Here we can see three different objectives with different numerical aperture. Numerical aperture is measured as the index of refraction of the material between the objective and the speciment *(if it's air then index of refraction is 1; for oil it is higher!)* and it's index of refractions times the sinus of the half-angle between a normal vertical light $\alpha$ ; which means $$NA=n \cdot sin(\alpha)$$ .  | Difference between high and low NA is how much the spherical cap is collected, and therefore how much of the spherical cap exists converging back to that point. |

| ![na](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/na.png) | ![wavelength](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/wavelength.png) | 
| ---- |---- |
| The final image is related to fringe period. Any wavelets that closer together can generate bigger structure, so it's related to how fine this fringe period can be. | The other thing what determines the quality is the wavelength of the light. | 

| ![three_wavelets](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/three_wavelets.png) | ![five_wavelets](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/five_wavelets.png) | ![more_wavelets](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/more_wavelets.png) | ![morewavelets](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/morewavelets.png) |
| --- | --- | --- | --- |


| ![color_light_wavelength](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/color_light_wavelength.png) | ![airydisk](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/airydisk.png) | 
| ---- |---- | 

Airy disk equation is about the center of the light and the first black ring: $\frac{0.61 \cdot \lambda}{NA}$ , where $\lambda$ is the wavelength of the light; $NA$ is the numerical aperture. More spread along the optical axis, (z axis), first dark island at $\frac{2n \cdot \lambda}{NA^2}$ where $n$ is the index of refraction *(the medium of objective and specimen)*, the else is the same. We can see that the numerical aperture can create much more problems with the depth of imaging then in the plane.

![na_depth](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Point%20Spread%20Function%20(Jeff%20Lichtman)/na_depth.png)

## Resolution of a Microscope (Jeff Lichtman, Harvard University)
> [lecture site](https://www.ibiology.org/talks/resolution-of-a-microscope/) | [YouTube](https://www.youtube.com/watch?v=n2asdncMYMo) | [lumenlearning.com - Physics, wave optics, *Limits of Resolution: The Rayleigh Criterion*](https://courses.lumenlearning.com/suny-physics/chapter/27-6-limits-of-resolution-the-rayleigh-criterion/)
>
> The resolution of a microscope can be defined as the smallest distance at which two small objects can still be seen as separate objects. This lecture discusses various criteria for resolution, the factors that influence resolution in the lateral and axial planes, and how to sample an image adequately using a camera or confocal microscope, such that the full optical resolution is retained.

This question is not about how close they are in real space, it is about how close they are in image space and still we can resolve as separate objects. There is no generally-accepted criterion but most microscopists use the Rayleigh criterion, partly because the distance between two points is $\frac{0.61 \lambda}{NA} $ if we talk about the lateral criterion. If we talk about the depth criterion in xz or yz then this is $\frac{2n \lambda}{NA^2}$ 

| ![rayleighcriterion_lateralanddepth](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/rayleighcriterion_lateralanddepth.png) | ![image_of_small_specimen](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/image_of_small_specimen.png) |
| --- | --- |


> ### Rayleigh criterion
>
> [The Rayleigh criterion](http://hyperphysics.phy-astr.gsu.edu/hbase/phyopt/Raylei.html) is the generally accepted criterion for the minimum resolvable detail - the imaging process is said to be diffraction-limited when the first diffraction minimum of the image of one source point coincides with the maximum of another.
> | ![diffraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/diffraction.jpg) | ![Rayleigh criteria](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/rayc.gif) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Figure_28_06_02a.jpg) |
> | ----- | ----- | ----- |
> 
> | $$sin \theta_R = \frac{\lambda}{d}$$ | $$sin \theta_R = 1.22 \frac{\lambda}{d}$$ |
> | ----- | ----- |
> | Single slit | Circular aperture |
> 
> If all parts of an imaging system are considered to be perfect, then the resolution of any imaging process will be limited by diffraction. Considering the single slit expression above, then when the wavelength is equal to the slit width, the angle for the first diffraction minimum is 90°. This means that the wave is spread all the way to the plane of the slit and will not contain resolvable information about the source of the wave. This leads to the simplified statement that the limit of resolution of any imaging process is going to be on the order of the wavelength of the wave used to image it.

### Aliasing

| ![aliasing1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/aliasing1.png) | ![aliasing2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/aliasing2.png) | ![aliasing3](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/aliasing3.png) |
| ---- | --- |---- |

This can be solved if we increase the number of pixels, but what should be the exact number? This is told by [Nyquist limit](https://imb.uq.edu.au/research/facilities/microscopy/training-manuals/microscopy-online-resources/image-capture/nyquist-conditions), which  *"states that a periodic signal must be sampled at more than twice the highest frequency component of the signal. In imaging, we recommend sampling at least 2.3X the highest frequency."* : $0.3 \lambda / NA$ . 

| ![sampling](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/sampling.png) | ![samplingsolved](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/samplingsolved.png) | ![samplingexample](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/samplingexample.png) | ![solution](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Resolution%20(Jeff%20Lichtman)/solution.png) |
| ------ | ---- | ------ |  ------ | 


> | ![cell moving](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/nyquist1.png) ![lower resolution](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/nyquist2.png) | ![resolution settings](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/nyquist3.png) |
> | --- | --- |
> | Let's look at this first example where we collect an image over time. In other words, for a cell with diameter of 10 um that moves at 5 um/ minute, calculation: $$\frac{10 um}{2.3 \cdot 5 \frac{um}{m}} = \text{52 secs}$$ Therefore, we need to sample the data with a time resolution of a minimum of 52 seconds to be able to capture the movement correctly. Here, we are able to visualize the cell moving from the top left corner, to the bottom right corner and reliably determine it is the same cell. However, if we collect an image every 2 minutes, we cannot determine if this is the same cell, or if these are 2 different cells. However, if instead of tracking the entire cell, we are interested in tracking the nucleus which is 5 um in diameter, then we would need to collect another image every 26 seconds. $$\frac{5um}{2.3 \cdot 5 \frac{um}{min}} = \text{26 sec}$$ | Likewise, if the width of your cell is 10 um, you need XY resolution to be less than 5um to be able to resolve 2 cells next to each other. |
>
> *Resolution is the ability of an objective to definitively resolve an object, or group of objects, as a single entity(s). Magnification plays no part in the ability of an objective to resolve an object of a given size. While magnification will make an image bigger, it will not necessarily make it clearer. Resolution is governed solely by the Numerical Aperture (NA) of the objective, the wavelength of light and for axial resolution, the refractive index of the sample. Magnification will affect how much light the objective can collect. Higher magnification objectives collect less light, so where possible use the highest NA, lowest magnification objective available.*
>
> *It is important when discriminating small structures that the [Nyquist Sampling Criteria](https://imb.uq.edu.au/research/facilities/microscopy/training-manuals/microscopy-online-resources/image-capture/nyquist-conditions) is satisfied. Nyquist Sampling dictates that to optimally represent an analogue signal in digital space, **the analogue signal needs to be sampled at least 2.3 times. In microscopy terms this means that the pixel size of an image needs to be at least 2.3 times smaller than the object that is being resolved.***
>
> **[Microscopy Nyquist rate and PSF calculator](https://svi.nl/Nyquist-Calculator)**

## Bonus: Photons: What is Light? (Bo Huang, UCSF)
> [lecture site](https://www.ibiology.org/talks/photons/)  | [YouTube](https://www.youtube.com/watch?v=VzThe929HrU) | [Maxwell's Equations and Light Waves](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/lecture03_1.pdf)
>
> This lecture discusses the many aspects of light. Light can be described as rays, waves, or particles. Various aspects of all three of these descriptions are presented, such as ray tracing, wavelength, frequency, refraction, dispersion, diffraction, interference, the photoelectric effect and photons as the quantized energy of light.

### Light is a ray

It is easy to think about the light as rays. We can explain a lot phenomena with this idea, such as focal and image length; focal point; lens maker equation or ray tracing.

| ![focallength](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/focallength.png) | ![lensmakers_equation](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/lensmakers_equation.png) | ![raytracing](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/raytracing.png) |
| --- | --- | --- |

### Light is a wave

What we can't explain with the idea of rays we can explain as waves and the oscillation of waves. A wave has a few descriptor, such as wavelength $\lambda$ which is a the length between two oscillation or the number of oscillations, $\nu$ which is the frequancy. These sets the speed of the light $c=\nu \lambda = \text{2992792458 m/s in vacuum}$ .If we want to see it in other medium then we can calculate like this: $n=\frac{c}{\text{speed in the medium}} = \frac{ \lambda_{vacuum}}{ \lambda_{medium}}$ This is the refractive index, which tell you the direction of the light based on [Snell's law](https://en.wikipedia.org/wiki/Snell%27s_law): $n_1 sin \theta_1 = n_2 sin \theta_2$ .

| ![wavelength](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/wavelength.png) | ![refractive_index.png](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/refractive_index.png) | ![explainingrefraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/explainingrefraction.png) | ![sinusoidal wave](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Sine_voltage.svg.png) |
| ---- | ---- | ---- | ---- | 

Based on this we can add waves to each other, this is interference. If the distance betwee n two added waves peak position is enough small $\Delta \varphi$ the waves will help each other and it will be constructive interference. If not than we call destructive interference. Of course if we have a thin glass than the refraction depends on the wavelength and as wee see the depth of the glass as well. But in most cases we don't have such pure interference. 

| ![interference](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/interference.png) | ![constructive_interference](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/constructive_interference.png) | ![destructive_interference](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/destructive_interference.png) | ![examplerefraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/examplerefraction.png) |
| ---- | --- | ---- | ---- |

$$\text{light propagation = diffraction + interference}$$

In a diffraction if we have an incoming plane wave then after the diffraction in reality we have plane waves again because of the interference, because a plane wave technically is a lot of spherical waves. To describe this if we have a point after the aperture, and all kind of light waves are coming to this point, each of them has [amplitude](https://en.wikipedia.org/wiki/Amplitude) $A$ with travel distance $l$. So the amplitude at total at this point will be $\sum A_i sin(\frac{2 \pi l_i}{\lambda})$ which makes the intensity in this point $I = A^2$ .

| ![interferenceanddiffraction](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/interferenceanddiffraction.png) | ![interference_example](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/interference_example.png) | ![double_slit_interference](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/double_slit_interference.png) | ![double_slit_interference2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/double_slit_interference2.png) | 
| ---- | ---- | ---- | ---- |

### Light is a particle of an electromagnetic wave
To understand this we need [Maxwell's equations](https://en.wikipedia.org/wiki/Maxwell%27s_equations): 

$$\nabla \cdot E = \frac{ \rho }{ \varepsilon_0 }$$
Coulumb's law says charges can generate static electric field. The electric field is proportional with the charge density.

$$\nabla \cdot B = 0$$
The magnetic force lines forms circles. There is no charge for magnetic fields. 

$$\nabla \cdot E = \frac{\delta B}{ \delta t}$$
Changing magnetic field generates electric field.

$$\nabla \cdot B = \mu_0J + \mu_0 \varepsilon_0 \frac{\delta E}{ \delta t}$$
The electric current and changing electric field generate magentic field.

The light is an electric field which generates a magnetic field which generates a magnetic field....

We can see this if we use a polarizer because these waves are perpendicular to each other.

| ![light_as_elevtromagnetic_field](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/light_as_elevtromagnetic_field.png) | ![polarization](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/polarization.png) | ![quantized_energy_of_light](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/quantized_energy_of_light.png) | ![wave_particle_duality](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20What%20Is%20Light%20(Bo%20Huang)/wave_particle_duality.png) |
| ----------- | ------------ | ------------ | ------------ |

## Bonus: Fourier Space: Fourier Transform (Bo Huang, UCSF)
> [course link](https://www.ibiology.org/talks/fourier-transform/) | [YouTube](https://www.youtube.com/watch?v=xhO8iz2qCOE)
>
> The Fourier transform is intimately associated with microscopy, since the alternating planes occurring in the microscope (focal plane – back-focal plane, etc.) are related to each other by a function very similar to the Fourier transform. This lectures explains the Fourier transform in terms understandable to non-mathematicians, and explains the relations with microscopy.

We have $I(r) = A sin(k \cdot r + \varphi_0$ and  $F(x) = \sum G(x) sin(k x)$. Fourier transform is about to cconvert one sinusoid function to a squered function. This is the way how e transform an image into a Furier image which is in frequency space. We can do the inverse direction too.

| ![convert](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/convert.png) | ![furierspace](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/furierspace.png) | ![cuttedaroundcenter](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/cuttedaroundcenter.png) | ![centercutted](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/centercutted.png) | ![fourier_transform_objective_lens](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/fourier_transform_objective_lens.png) |
| ---- | ----- |---- | ----- |----- |

Furier transform of light intesity and the phase, at the aperture is the point spread function of the microscope.

| ![resolution](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/resolution.png) | ![psf](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fourier%20Space%20(Bo%20Huang)/psf.png) |
| --- | ---- |
  
## Darkfield and Phase Contrast Microscopy (Edward Salmon, University of North Carolina)
> [lecture site](https://www.ibiology.org/talks/phase-contrast-microscopy/) | [YouTube](https://www.youtube.com/watch?v=I4ZQm-CAgL8) | [C Robert Bagnell, Jr., Ph.D., 2012 - Phase Contrast](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/lm-ch-10-phase-contrast.pdf) | [Optical Pathways in the Phase Contrast Microscope](https://www.microscopyu.com/tutorials/optical-pathways-in-the-phase-contrast-microscope)
> 
> This lecture describe the principles of dark field and phase contrast microscopy, two ways of generating contrast in a specimen which may be hard to see by bright field. The lecture describes how the phase rings work to generate interference between the diffracted and undiffracted light.

The purpose of darkfield microscopy is what we can see on the second image, and it's working like as it is showed on the fst image. Darkfield illumination is where cone of light is with $NA_{cond}>NA_{obj}$ ; objective iris prevents light entering objective. The scattered light generated by light field being focused by the speciement is collected by the objective and it is focused on the image plane. Here we can say that high NA is high resolution. 

| ![structure](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/structure.png) | ![darkfield](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/darkfield.png) | ![conclusion](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/conclusion.png) | ![illumination](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/illumination.png) | ![highNAmenashighresolution](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/highNAmenashighresolution.png) |
| --- | --- | --- | ---- | ---- |

Advantages of darkfield :
- high sensitivity - black background provides high contrast to see scattered light for small objectives
- excellent fro low magnification outlines of individual cells

Disatvantages of darkfield: 
- resolution limited by need for $NA_{cond}>NA_{obj}$
- scattered light in thick specimens lowers contrast of fine structural detail
- poor dept of field
- often inaccurate images because missing interference with undiffracted light
- special light sources

| ![phasecontrast](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phasecontrast.png) | ![wavesfromthespeciment](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/wavesfromthespeciment.png) | ![diffractionindarkfield](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/diffractionindarkfield.png) | ![dark_interference](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/dark_interference.png) | ![thereisnoNA](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/thereisnoNA.png) |
| --- | --- | ---- |---- |---- |

Based on the speciment light is generated with the interference of diffracted and undiffracted light. In phase contras we use [Annular illumination](https://www.sciencedirect.com/topics/engineering/annular-illumination) method what is a method of apertured exposure in a wafer stepper that changes the optical pathway and incident landing angle of incoming light. The condenser annulus is typically constructed as an opaque flat-black (light absorbing) plate with a transparent annular ring, which is positioned in the front focal plane (aperture) of the condenser so the specimen can be illuminated by defocused, parallel light wavefronts emanating from the ring. The microscope condenser images the annular diaphragm at infinity, while the objective produces an image at the rear focal plane. 

The phase ring:
- dimaeter: 50% of $NA_{obj}$
- condenser annulus diameter must be match to phase ring!
- condenser annulus must be aligned with phase ring in the objective

| ![condenser annulus](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Screenshot%202024-01-18%20at%2011-34-29%20Optical%20Pathways%20in%20the%20Phase%20Contrast%20Microscope.png) | ![phasering](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phasering.png) |  ![phase_microscope](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phase_microscope.png) | ![phases](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phases.png) | ![phasering](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phasering.png) |
| --- | --- | --- | --- |--- |

| ![focal_plane](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/focal_plane.png) | ![phase1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phase1.png) | ![phase2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phase2.png) | ![phase3](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/phase3.png) |
| --- | ---- | ---- | --- |

Conclusion: the most important parameter in the basic design of a phase contrast microscope is to isolate the surround and diffracted light waves emerging from the specimen so that they occupy different locations in the diffraction plane at the rear aperture of the objective. Explores light pathways through a phase contrast microscope and dissects the incident electromagnetic wave into surround (S), diffracted (D), and resultant (particle; P) components.

| ![brightfield](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/brigthfieldphasecontrast.png) | ![negative phase contrast](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/negativephasecontrast.png) | ![positivephasecontrast](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/positivephasecontrast.png) |
| --- | ---- | --- |

Usage in [Epi-flourescence mmicroscopy](https://www.thermofisher.com/ro/en/home/life-science/cell-analysis/cell-analysis-learning-center/molecular-probes-school-of-fluorescence/imaging-basics/fundamentals-of-fluorescence-microscopy/epifluorescence-microscope-basics.html). 

| ![epiimage](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/epiimage.png) | ![epiflourescence](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/epiflourescence.png) | ![epimic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon)/epimic.png) | ![optical train](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/epiflourescencemicopticaltrain.gif) |
| ---- | ---- | ---- | --- |

## Polarized Light and its Interaction with Material (Shinya Inoue, Marine Biological Laboratory)
> [course site](https://www.ibiology.org/talks/polarized-light/) | [YouTube](https://www.youtube.com/watch?v=_-C2Db-aBuU)
>
> Shinya Inoue, one of the great innovators of polarization microscopy, describes what polarized light is and how light passing through a calcite crystal is split into two orthogonal polarized light beams.

| ![calcyte](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Crystal_on_graph_paper.jpg) | ![polar filter](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Calcite_and_polarizing_filter.gif) | ![polarization](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Positively_birefringent_material.svg.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Polarized%20Light%20and%20its%20Interaction%20with%20Material%20(Shinya%20Inoue)/Screenshot%202024-01-18%20at%2012-56-23%20Microscopy%20Darkfield%20and%20Phase%20Contrast%20Microscopy%20(Edward%20Salmon).png) |
| --- | ---- | ---- | --- |

## Polarization Microscopy (Edward Salmon, University of North Carolina)
> [Course site](https://www.ibiology.org/talks/polarization-microscopy/) | [YouTube](https://www.youtube.com/watch?v=Ph3AdDUG9vI) | [microscopyU - polarized light microscopy](https://www.microscopyu.com/techniques/polarized-light/polarized-light-microscopy) | [microscopyU -  Principles of Birefringence](https://www.microscopyu.com/techniques/polarized-light/principles-of-birefringence) | [Introduction to Biological Polarization Microscopy(Part 2)](https://micro.magnet.fsu.edu/primer/techniques/polarized/biologicalparttwo.html)
>
> Polarization microscopy probes the interaction of molecules with polarized light and is particularly good for examining well-order structures composed of polymers, such as the mitotic spindle. This lecture describes the components of a polarization microscope (e.g. polarizer, analyzer), birefringence and how it is exploited to generate images, adjusting a polarization microscope, examples of images, and new methods such the LC-Polscope.

| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/polarlightmic.jpg) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Interference%20Colors%20in%20Gout%20and%20Pseudo-Gout%20Crystals.png) |
| --- | --- |


Polarized light is a contrast-enhancing technique that improves the quality of the image obtained with birefringent materials when compared to other techniques such as darkfield and brightfield illumination, differential interference contrast, phase contrast, Hoffman modulation contrast, and fluorescence. Polarized light microscopes have a high degree of sensitivity and can be utilized for both quantitative and qualitative studies targeted at a wide range of anisotropic specimens. Qualitative polarizing microscopy is very popular in practice, with numerous volumes dedicated to the subject. In contrast, the quantitative aspects of polarized light microscopy, which is primarily employed in crystallography, represent a far more difficult subject that is usually restricted to geologists, mineralogists, and chemists. However, steady advances made over the past few years have enabled biologists to study the birefringent character of many anisotropic sub-cellular assemblies.


| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Separation%20of%20Light%20Waves%20by%20a%20Birefringent%20Crystal.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Birefringent%20Calcite%20Crystal%20Electric%20Vector%20Orientations.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Light%20Path%20Through%20a%20Calcite%20Crystal.png) |
| --- |--- | --- |

[Birefringence](https://www.microscopyu.com/techniques/polarized-light/principles-of-birefringence) is formally defined as the double refraction of light in a transparent, molecularly ordered material, which is manifested by the existence of orientation-dependent differences in refractive index. Many transparent solids are optically isotropic, meaning that the index of refraction is equal in all directions throughout the crystalline lattice. 

| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Polarization%20(Edward%20Salmon)/polarizingmicroscope.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Polarization%20(Edward%20Salmon)/polarizedvsunpolarizedlight.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Polarization%20(Edward%20Salmon)/planepolarizedlight.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Polarization%20(Edward%20Salmon)/phasecontrastvspolarizing.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Polarization%20(Edward%20Salmon)/birefringence.png) |
| --- | --- |--- | --- | --- |


Images by polarization microscopes:

| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Natural%20and%20Synthetic%20Polymers%20in%20Polarized%20Light.jpg) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Oolite%20Thin%20Section%20in%20Polarized%20Light.jpg) |
| --- | --- |

## Differential Interference Contrast (DIC) Microscopy (Edward Salmon, University of North Carolina)
> [course site](https://www.ibiology.org/talks/differential-interference-contrast/) | [YouTube](https://youtu.be/FUa1GTc69y4?si=MoKqW7FQ5zzdnw4w) | [microscopyU - Differential Interference Contrast](https://www.microscopyu.com/techniques/dic) | [scientifica - A guide to Differential Interference Contrast (DIC)](https://www.scientifica.uk.com/learning-zone/differential-interference-contrast) |[microscopyU - Comparison of Phase Contrast & DIC Microscopy](https://www.microscopyu.com/tutorials/comparison-of-phase-contrast-and-dic-microscopy) | [DIC Microscope Configuration and Alignment](https://www.olympus-lifescience.com/en/microscope-resource/primer/techniques/dic/dicconfiguration/) | [DIC Microscopy Literature References](https://www.microscopyu.com/references/dic)
>
> Differential Interference Contrast (sometimes known as Normarski microscopy) is a variation of polarization microscopy which generates a high contrast “shadow” image of a specimen. The mechanism of the DIC (Wollaston) prisms is discussed along with how to generate optimal contrast.

[Differential Interference Contrast (DIC)](https://www.scientifica.uk.com/learning-zone/differential-interference-contrast) is a microscopy technique that introduces contrast to images of specimens which have little or no contrast when viewed using brightfield microscopy. The images produced using DIC have a pseudo 3D-effect, making the technique ideal for electrophysiology experiments. Using DIC produces high resolution images with good contrast. It is best for visualising unstained samples. When viewing fluorescently labelled compounds, using DIC reduces the fluorescence intensity slightly and so decreases the quality of fluorescence images. However, this decrease is not significant. As an Infrared (IR) light source is usually used, DIC is ideal for imaging thick specimens such as brain slices, as IR light penetrates deeper into tissues than visible light, due to its longer wavelength. An advantage of using DIC over other contrast techniques, such as phase contrast or oblique contrast is that in DIC the full aperture of the microscope is used. For example, in phase contrast, the annulus of the condenser restricts the aperture, reducing the resolution of the image. Unlike phase contrast, DIC images are not disturbed by halo artefacts.

| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/DIC-light-path-1.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/scope-only-labelled_180222_141156.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/dicalignmentfigure1.jpg) |
|---- | --- |--- |

**How does Differential Interference Contrast work?**

In DIC, light emitted from the source is linearly polarised by passing through a polariser. The linearly polarised beam of light enters an objective-specific prism, which splits it into two rays that vibrate perpendicular to each other. The rays are parallel as they pass through a condenser, but as they are vibrating perpendicular to each other, they are unable to cause interference.

The split beams pass through the specimen. The specimen’s varying thickness and refractive indices alter the wave paths of the beams. They then enter the objective, where they are focussed above the rear focal plane. The two beams enter a second prism, in the nosepiece, which combines them. Because the beams passed through different parts of the specimen, they have different lengths.

The analyser, which is a second polariser, brings the vibrations of the beams into the same plane and axis, causing destructive and constructive interference to occur between the two wavefronts. The light then travels to the eyepiece or camera, where a DIC image with differences in intensity and colour, can be seen. 

The most fundamental distinction between differential interference contrast ([DIC](https://www.microscopyu.com/tutorials/comparison-of-phase-contrast-and-dic-microscopy)) and phase contrast microscopy is the optical basis upon which images are formed by the complementary techniques. Specimens examined by these contrast-enhancing methods produce images that are often quite different in appearance and character when objectively compared. This interactive tutorial explores many of the similarities and differences exhibited between images captured with phase contrast and DIC microscopy.

| ![differences](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/differences.png) | ![feature1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/feature1.png) | ![features2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/features2.png) | ![features3](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/features3.png) |
|---- | ---- | ---- | --- |
| ![dicprism](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/dicprism.png) | ![dicmicstruct](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/dicmicstruct.png) | ![divergent](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/divergent.png) | ![intensity](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/intensity.png) |

| ![intensitychanges](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/intensitychanges.png) | ![maxcontrast](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Differential%20Interference%20Contrast%20(DIC)%20Microscopy%20(Edward%20Salmon)/maxcontrast.png) |
| --- | --- |

## Bonus: Examples of Using Polarization Microscopy (Shinya Inoue, Marine Biological Laboratory)
> [course link](https://www.ibiology.org/talks/examples-of-using-polarization-microscopy/) | [YouTube](https://www.youtube.com/watch?v=VEl0-VRg_ow)
>
> Shinya describes examples of using polarization microscopy to image living biological organisms, including watching processes of mitosis.

| ![polarization_rectifier](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Examples%20of%20Using%20Polarization%20Microscopy%20(Shinya%20Inoue)/polarization_rectifier.png) |![compensator](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Examples%20of%20Using%20Polarization%20Microscopy%20(Shinya%20Inoue)/compensator.png) | ![spermstructure](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Examples%20of%20Using%20Polarization%20Microscopy%20(Shinya%20Inoue)/spermstructure.png) |   
| --- | --- |--- | 

## Bonus: Pragmatics of DIC and Video-Enhanced Contrast Microscopy (Edward Salmon, University of North Carolina)
> [course site](https://www.ibiology.org/talks/video-enhanced-contrast-microscopy/) | [YouTube](https://www.youtube.com/watch?v=mdY__N7GGgY) | [Biocyclopedia - Video Enhanced Contrast Microscopy](https://biocyclopedia.com/index/cell_biology_methods/video_enhanced_contrast_microscopy.php) | [Dieter G Weiss - Video-Enhanced Contrast Microscopy - Chapter 6](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/dieter_g_weiss_2006-VCM.pdf)
>
> In this lecture titled “Pragmatics of DIC and Video-Enhanced Contrast Microscopy,” Ted Salmon extends the beginning DIC lecture by discussing how to align the DIC microscope, how color DIC is produced, how to use video-enhancement to visualize faint objects such as microtubules and how to combine DIC with fluorescence.

| ![missmatchedprism](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/missmatchedprism.png) | ![poorcondenser](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/poorcondenser.png) | ![matched](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/matched.png) | ![colordic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/colordic.png) |
| --- | ---- |--- | --- |
|  ![highres](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/highres.png) | ![highres2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/highres2.png) | ![timeres](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/timeres.png) | ![microtube](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/microtube.png) |

| ![dicfluorescencemultimodemic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/dicfluorescencemultimodemic.png) | ![multimodedicimage](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Pragmatics%20of%20DIC%20and%20Video-Enhanced%20Contrast%20Microscopy%20(Edward%20Salmon)/multimodedicimage.png) |
| --- | --- |

## Introduction to Fluorescence Microscopy (Nico Stuurman, UCSF/HHMI)
> [course site](https://www.ibiology.org/talks/fluorescence-microscopy/) | [YouTube](https://youtu.be/AhzhOzgYoqw?si=G_aCJtExYv7dYvaY) | [MicroscopyU - Introduction to Fluorescence Microscopy](https://www.microscopyu.com/techniques/fluorescence/introduction-to-fluorescence-microscopy) | [M. J. Sanderson, I. Smith, I. Parker, M. D. Bootman - Fluorescence Microscopy,  doi: 10.1101/pdb.top071795](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4711767/) | [George Rice - Fluorescent Microscopy](https://serc.carleton.edu/microbelife/research_methods/microscopy/fluromic.html) | [Wikipedia](https://en.wikipedia.org/wiki/Fluorescence_microscope) | [What is the Stokes Shift?](https://www.edinst.com/blog/what-is-the-stokes-shift/) | [Olympus - Interference Filters](https://www.olympus-lifescience.com/en/microscope-resource/primer/java/filters/interference/) 
>
> Fluorescence is a process in which matter absorbs light and re-emits at a different wavelength. Fluorescence is widely used in biological microscopy. This lecture describes the principles of fluorescence and fluorescence microscopy.

Fluorescence is when we light something with a light what has shorter wavelenght and the outcome is a longer wavelength with lower energy. The absorption and subsequent re-radiation of light by organic and inorganic specimens is typically the result of well-established physical phenomena described as being either fluorescence or phosphorescence. The emission of light through the fluorescence process is nearly simultaneous with the absorption of the excitation light due to a relatively short time delay between photon absorption and emission, ranging usually less than a microsecond in duration. When emission persists longer after the excitation light has been extinguished, the phenomenon is referred to as phosphorescence. Here we can see the [Stokes Shift](https://www.edinst.com/blog/what-is-the-stokes-shift/) which is the spectral shift to lower energy between the incident light and the scattered or emitted light after interaction with a sample.

> In fluorescence spectroscopy, the Stokes shift is the difference between the spectral position of the maximum of the first absorption band and the maximum of the fluorescence emission and can be expressed in either wavelength or wavenumber units as shown in Figure.
>
> *It should be noted that the wavenumber Stokes shift expression written above is actually only an approximation since it assumes that the wavenumber maxima are at the same position as the wavelength maxima which is not strictly true.*
>
> The origin of the Stokes shift is commonly represented in a [Perrin-Jablonski diagram](https://www.edinst.com/blog/jablonski-diagram/) as an initial excitation to a higher vibrational level of the $S_1$ followed by a rapid non-radiative decay to the vibrational ground state of the $S_1$ *(4th Figure)*, which means that **the fluorescence will have a lower energy than the absorbed photon and therefore a longer wavelength**.
>
> In the quantum mechanical picture of the Franck-Condon principle, the difference in mass between the nuclei and the electrons allows the wavefunction of the molecular state to be separated into an electronic component and a nuclear (vibrational) component (Born-Oppenheimer Approximation). The transition probability between each vibrational level of the $S_0$ and $S_1$ is then proportional to the overlap integral between their vibrational wavefunctions.
>
> Combining these two descriptions a hybrid classical-quantum picture of the concept can be drawn on *5th Figure*.

| ![fluorescenceemissionlight.png](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20-%20Introduction%20to%20Fluorescence%20Microscopy%20(Nico%20Stuurman)/fluorescenceemissionlight.png) | ![fluorescencespectrum](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20-%20Introduction%20to%20Fluorescence%20Microscopy%20(Nico%20Stuurman)/fluorescencespectrum.png) | ![jablonski](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20-%20Introduction%20to%20Fluorescence%20Microscopy%20(Nico%20Stuurman)/jablonski.png) |
| ---- | --- | --- | 
| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Stokes-shift-2-768x630.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/stokes-shift-4-768x922.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/HarmonicOscillator-614x1024.png) |

> The fluorescence emission that results from the light flux discussed above depends on the absorption and emission characteristics of the fluorophore, its concentration in the specimen, and the optical path length of the specimen. In mathematical terms, the fluorescence produced $F$ is given by the equation: $$F = \sigma \times Q \times I$$ where $\sigma$ is the molecular absorption cross-section, $Q$ is the quantum yield, and $I$ is the incident light flux (as calculated above). Assuming that fluorescein is the fluorophore, the absorption cross-section ($\sigma$) is $3 \times 10 \times E(-16)$ square centimeters per molecule, $Q$ equals $0.99$, resulting in a value for $F$ of 100,000 photons per second per molecule.

In the microscopy we use filters to filter the exciting light, to get only the wavelength what excite the dye *(pigment)* and then the fluerscence light come back from the same objective to the eye. The most important part of this type of microscope is the filter. Nowdays we use [interference filters](https://www.vision-doctor.com/en/interference-filters.html). A thin layer which is half thin of thte wavelength of the light with semireflective coating between. THis reflect the desctructive interference and transmit the constructive interference.  

[Dielectric coatings are often bundled into units termed cavities](https://www.olympus-lifescience.com/en/microscope-resource/primer/java/filters/interference/), which are constructed of three to five alternating salt and metal oxide (and sometimes pure metal) layers separated by a wider layer of magnesium fluoride termed a spacer (see Figure). The spacers are produced with a thickness that corresponds to even multiples of a quarter or half wavelength in order to reflect or transmit light in registration with the dielectric layers. Increasing the number of cavities utilized to build an interference filter produces a proportional increase in the slope of the cut-on and cut-off wavelength transmission boundaries. Filters featuring up to 15 stacked cavities can have a total of over 75 individual dielectric layers and display bandwidths only a few nanometers in size.

| ![microscopestructure](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20-%20Introduction%20to%20Fluorescence%20Microscopy%20(Nico%20Stuurman)/microscopestructure.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Screenshot%20Introduction%20to%20Fluorescence%20Microscopy-microscope.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Screenshot%20Introduction%20to%20Fluorescence%20Microscopy-filter.png) | ![interferencefilters](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20-%20Introduction%20to%20Fluorescence%20Microscopy%20(Nico%20Stuurman)/interferencefilters.png) |
| ---- | --- | ---- | --- |
| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/principle_destructive_interference.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/interferencejavafigure1.jpg) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/interferencejavafigure2.jpg) | ![fasterselection](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20-%20Introduction%20to%20Fluorescence%20Microscopy%20(Nico%20Stuurman)/fasterselection.png) |

## Fluorescent Probes: Organic Dyes and Quantum Dots (Timothy Mitchison, Harvard University)
> [course site](https://www.ibiology.org/talks/fluorescent-probes/) | [YouTube](https://www.youtube.com/watch?v=gSGq8gOLXwY)
>
> In this lecture about fluorescent probes, Timothy Mitchison describes the chemical properties of organic dyes and quantum dots (Q dots) that make them fluoresce and how their fluorescence wavelength is tuned. It also describes how dyes can be attached to proteins, photobleaching, and other dyes that bind DNA (Hoechst) or organelles (mitotracker, lysotracker).

| ![homo-lumo](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/homo-lumo.png) | ![largerconj](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/largerconj.png) | ![visiblespectrum](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/visiblespectrum.png) | ![qdot](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/qdot.png) |
| ---- | ---- | ---- | ---- |
| ![qdotemission](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/qdotemission.png) | ![photostability](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/photostability.png) | ![photodamage](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/photodamage.png) | ![imprpobes](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/imprpobes.png) |
| ![phform](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/phform.png) | ![thiolreactive](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/thiolreactive.png) | ![unproton](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/unproton.png) | ![snap](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/snap.png) |
| ![snaptag](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/snaptag.png) | ![flash](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/flash.png) | ![rohadamile](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/rohadamile.png) | ![mitotracker](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Fluorescent%20Probes%20(Timothy%20Mitchison)/mitotracker.png) |

## Fluorescent Proteins and the Story Behind GFP (Roger Tsien, UCSD/HHMI)
> [course site](https://www.ibiology.org/talks/fluorescent-proteins/) | [YouTube](https://www.youtube.com/watch?v=qK9aYnkIr3w) | [ EFGP and DsRed expressing cultures of Escherichia coli imaged by confocal, two-photon and fluorescence lifetime microscopy; doi:  10.1016/s0014-5793(00)01896-2](https://pubmed.ncbi.nlm.nih.gov/10981721/) /[Sci-Hub](https://sci-hub.se/10.1016/s0014-5793(00)01896-2) / [file](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/jakobs2000.pdf) 
>
> Live cell imaging has been revolutionized by the discovery of the green fluorescent protein (GFP). Roger Tsien talks about fluorescent proteins and covers the history of GFP, how it folds and becomes fluorescent, how it has been mutated to produce additional colors (blue, cyan, yellow). It also covers novel photoswitchable and photoactivatible fluorescent proteins, whose color can be changed by light, and new infrared fluorescent proteins.

The wide spectrum of [fluorescent proteins](https://www.microscopyu.com/techniques/fluorescence/fluorescent-protein-imaging-parameters) and derivatives uncovered thus far are quite versatile and have been successfully employed in almost every biological discipline from microbiology to systems physiology. These unique probes have proven extremely useful as reporters for gene expression studies in both cultured cells and entire animals. In living cells, fluorescent proteins are most commonly utilized to track the localization and dynamics of proteins, organelles, and other cellular compartments, as well as a tracer of intracellular protein trafficking. Quantitative imaging of fluorescent proteins is readily accomplished with a variety of techniques, including widefield, confocal, and multiphoton microscopy, to provide a unique window for exposing the intricacies of cellular structure and function.

| ![gfp](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/gfp.png) | ![mutationsofgfp](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/mutationsofgfp.png) | ![chrystalstructure](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/chrystalstructure.png) | ![colors](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/colors.png) |
| --- | --- | --- | --- |
| The fluorescence appears because of Oxigen, but creates $H_2O_2$ | The original jellyfish ଳ is `WT` | | ![wavelengthsdiffchromatic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/wavelengthsdiffchromatic.png) |
| ![fluorescentproteins](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/fluorescentproteins.png) | ![dsred](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/dsred.png) | ![problemswtihdsred](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/problemswtihdsred.png) | ![2004palette](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/2004palette.png) |

As we can see the main problem with the [DsRED](https://www.fpbase.org/protein/dsred/) is that it [changes the light and color during the time spontaneusly](https://en.wikipedia.org/wiki/Red_fluorescent_protein). That's why [Dendra protein](https://www.fpbase.org/protein/dendra2/) is much better.

| ![dendra](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/dendra.png) | ![reversible](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/reversible.png) | 
| --- |--- |

> ### [NIR](https://www.sciencedirect.com/topics/chemistry/nir-spectroscopy) for not tranparent objects
> [MicroscopyU - Imaging Fluorescent Proteins](https://www.microscopyu.com/techniques/fluorescence/fluorescent-protein-imaging-parameters) | [*Shcherbakova, D., Baloban, M., Emelyanov, A. et al.* Bright monomeric near-infrared fluorescent proteins as tags and biosensors for multiscale imaging. Nat Commun 7, 12405 (2016). https://doi.org/10.1038/ncomms12405](https://www.nature.com/articles/ncomms12405) | [Daria Shcherbakova - Quick Guide to Near-Infrared Fluorescent Proteins](https://blog.addgene.org/quick-guide-to-near-infrared-fluorescent-proteins) | [*Matlashov, M.E., Shcherbakova, D.M., Alvelid, J. et al.* A set of monomeric near-infrared fluorescent proteins for multicolor imaging across scales. Nat Commun 11, 239 (2020). https://doi.org/10.1038/s41467-019-13897-6](https://www.nature.com/articles/s41467-019-13897-6) | [*Feng Liu, Huimin Hu, et al.* ACS Sensors 2022 7 (7), 1855-1866 DOI: 10.1021/acssensors.2c00286  A Bright Monomeric Near-Infrared Fluorescent Protein with an Excitation Peak at 633 nm for Labeling Cellular Protein and Reporting Protein–Protein Interaction](https://pubs.acs.org/doi/10.1021/acssensors.2c00286)
> 
> | ![nir](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/nir.png) | ![tetrapyroll](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/tetrapyroll.png) | ![infrafluorescent](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/infrafluorescent.png) | ![ifp](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Proteins%20and%20the%20Story%20Behind%20GFP%20(Roger%20Tsien%2C%20UCSDHHMI)/ifp.png) | 
> | --- | --- |--- | --- |
>
> [Near-infrared fluorescent proteins (NIR FPs)](https://blog.addgene.org/quick-guide-to-near-infrared-fluorescent-proteins) were derived from natural photoreceptors incorporating biliverdin IXα (BV) autocatalytically. Fortunately for probe developers and users, BV is present in eukaryotic cells as an intermediate of heme metabolism (Shemetov et al, 2017). Modern state-of-the-art NIR FPs can be used similar to GFP-like probes by supplying a single gene to cells. Still in bacteria, which do not produce BV, an additional gene coding for heme oxygenase should be supplied (Baloban et al., 2017). 
>
> It is important to consider [effective brightness of NIR FPs in cells](https://pubmed.ncbi.nlm.nih.gov/26115447/), in addition to their molecular brightness (a product of the fluorescence quantum yield and the extinction coefficient) (Shcherbakova et al., 2015). The effective brightness reflects the concentration of BV-containing molecules in a cell and depends on the efficiency of BV incorporation by an apoprotein and protein expression level. State-of-the-art FPs (listed in Table 1) were optimized for efficient binding of endogenous BV. 
>
> When imaging NIR FPs, consider using light sources that produce enough light intensities in NIR. Usually, mercury lamps are suboptimal, metal halide lamps are better, and xenon lamps are the best for working in this spectral range. You can also check that no filters and/or cold mirrors block the NIR light coming from your source and to your camera. 
>
> #### Prospects of IFPs
> - Thousands more phytocromes to be explored
> - ~740nm excitation may of Pfr state may be possible
> - In vivo monitoring of phatogens
> - additional colors to orthogonal to existing colors
> - acceptor of [FRET](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2173363/) and [BRET](https://www.berthold.com/en/bioanalytic/knowledge/glossary/bret/)

## Fluorescent Protein Indicators (Roger Tsien, UCSD/HHMI)
> [course site](https://www.ibiology.org/talks/fluorescent-protein-indicators/) | [YouTube](https://www.youtube.com/watch?v=QoRTv2qZXbY) | [MicroscopyU - Introduction to Fluorescent Proteins](https://www.microscopyu.com/techniques/fluorescence/introduction-to-fluorescent-proteins) | [*Wenfeng Liu et al*, Genetically encoded single circularly permuted fluorescent protein-based intensity indicators; DOI 10.1088/1361-6463/ab5dd8](https://iopscience.iop.org/article/10.1088/1361-6463/ab5dd8) / [Sci-Hub](https://sci-hub.se/10.1088/1361-6463/ab5dd8) / [file](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/Genetically%20encoded%20single%20circularly%20permuted%20fluorescent%20protein-based.pdf)
>
> In this talk titled “Fluorescent Protein Indicators,” Roger Tsien discusses how fluorescent proteins have been turned into indicators for a wide variety of biological properties, including pH, redox potential, as well as for signaling molecules such as phosphoinositides. The talk also describes how reporters are used to measure the activity of enzymes like kinases, phosphatases, and proteases. It covers both single proteins whose intensity or wavelength changes, as well as reporters using Forster resonance energy transfer (FRET).

| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/1024px-GFP_structure.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/1xa9.jpg) | <img src="https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Green%20Fluorescent%20Protein%20Fluorophore%20Maturation.png" height="20%"></img> |
| --------------- | -------------------- | ----------------- |
| The green fluorescent protein [(GFP)](https://en.wikipedia.org/wiki/Green_fluorescent_protein) | Yellow fluorescent protein [(YFP)](https://en.wikipedia.org/wiki/Yellow_fluorescent_protein) | Among [the most important aspects of the green fluorescent protein](https://www.microscopyu.com/techniques/fluorescence/introduction-to-fluorescent-proteins) to appreciate is that the entire 27 kiloDalton native peptide structure is essential to the development and maintenance of its fluorescence. It is remarkable that the principle fluorophore is derived from a triplet of adjacent amino acids: the serine, tyrosine, and glycine residues at locations 65, 66, and 67 (referred to as Ser65, Tyr66, and Gly67; see last Figure). Although this simple amino acid motif is commonly found throughout nature, it does not generally result in fluorescence. What is unique to the fluorescent protein is that the location of this peptide triplet resides in the center of a remarkably stable barrel structure consisting of 11 beta-sheets folded into a tube. **It is important to note that the native green fluorescent protein fluorophore exists in two states. A protonated form, the predominant state, has an excitation maximum at 395 nanometers, and a less prevalent, unprotonated form that absorbs at approximately 475 nanometers. Regardless of the excitation wavelength, however, fluorescence emission has a maximum peak wavelength at 507 nanometers, although the peak is broad and not well defined.** |

[Fluorescent biosensors](https://www.addgene.org/fluorescent-proteins/biosensors/) are proteins designed so that their fluorescent properties are sensitive to the presence of specific molecules or other factors. Most often, a conformational change in the sensing region of the construct leads to a change in signal intensity of a circularly permuted fluorescent protein (cpFP) or Förster resonance energy transfer (FRET) efficiency of a pair of fluorescent proteins. Genetically encoded biosensors can be introduced into cells, tissues, or organisms for long-term imaging and can be designed to specifically localize to cellular compartments or organelles. 

[The discovery of green fluorescent protein](https://www.microscopyu.com/techniques/fluorescence/introduction-to-fluorescent-proteins) in the early 1960s ultimately heralded a new era in cell biology by enabling investigators to apply molecular cloning methods, fusing the fluorophore moiety to a wide variety of protein and enzyme targets, in order to monitor cellular processes in living systems using optical microscopy and related methodology. When coupled to recent technical advances in widefield fluorescence and confocal microscopy, including ultrafast low light level digital cameras and multitracking laser control systems, the green fluorescent protein and its color-shifted genetic derivatives have demonstrated invaluable service in many thousands of live-cell imaging experiments.

| ![fp12](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/fp12.png) | ![fp34](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/fp34.png) | 
| ------- |------------ |
| ![phsensitive](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/phsensitive.png) | ![cftr](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/cftr.png) |
| ![genericstructure](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/genericstructure.png) | ![twophoton](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/twophoton.png) |
| ![fretsensors](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/fretsensors.png) | ![geneticallyencodedfret](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/geneticallyencodedfret.png) |
| ![normalvsrestingbrain](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/normalvsrestingbrain.png) | ![camp](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/camp.png) |

| ![whataregood1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/whataregood1.png) | ![whataregood2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/whataregood2.png) | ![whataregood3](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Fluorescent%20Protein%20Indicators%20(Roger%20Tsien%2C%20UCSDHHMI)/whataregood3.png) |
| -------------- |-------------- | --------------- |


## Optical Sectioning and Confocal Microscopy (Kurt Thorn, UCSF)
> [course site](https://www.ibiology.org/talks/confocal-microscopy/) | [YouTube](https://www.youtube.com/watch?v=1Q5V442V7xI) | [zeiss campus - Optical Sectioning Microscopy](https://zeiss-campus.magnet.fsu.edu/articles/opticalsectioning/index.html) | [Jeremy Sanderson - Optical Sectioning and Confocal Microscopy](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/Optical%20Sectioning%20and%20Confocal%20Microscopy.pdf) | [Olympus - Photomultiplier Tubes](https://www.olympus-lifescience.com/en/microscope-resource/primer/digitalimaging/concepts/photomultipliers/) | [What Is Spinning Disk Confocal Microscopy?](https://www.photometrics.com/learn/spinning-disk-confocal-microscopy/what-is-spinning-disk-confocal-microscopy)
>
> This talk introduces confocal microscopy, and discusses optical sectioning, reconstruction of 3D images, and how the laser-scanning confocal microscope and spinning disk confocal microscope work.

| ![slices](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/slices.png) | ![conventionalproblem](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/conventionalproblem.png) | ![confocalproblem](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/confocalproblem.png) | ![fluorescenceproblem](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/fluorescenceproblem.png) |
| --- |---- | ---- | ---- |

| ![confocalsolution](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/confocalsolution.png) | ![imageraster](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/imageraster.png) |
| ------- | ---- |

| ![lightsource1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/lightsource1.png) | ![scanning](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/scanning.png) | ![confocalopticalpath](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/confocalopticalpath.png) | ![photomultipliertube](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/photomultipliertube.png) |
| ------- | ----------- | ------------ | ----------- |
| ![confocaldrawback](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/confocaldrawback.png) | ![spinningdiskconfocal](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/spinningdiskconfocal.png) | ![pinholesimage](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/pinholesimage.png) | ![outoffocusrejection](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/outoffocusrejection.png) |

> | [Spinning disk confocal microscopy (SDCM)](https://www.photometrics.com/learn/spinning-disk-confocal-microscopy/what-is-spinning-disk-confocal-microscopy) represents an alternative to laser scanning confocal microscopy (LSCM). Rather than a single pinhole, a SDCM has hundreds of pinholes arranged in spirals on an opaque disk (figure right), which rotates at high speeds. When spun, the pinholes scan across the sample in rows, building up an image. Using a spinning disk vastly improves the speed of image acquisition (allowing for imaging of fast dynamic processes and live specimens), and considerably reduces photo damage. | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/What-is-spinning-disk_2.png) |
> | ---- | --- |
> | The pinholes in the disk are arranged so that every part of the image is scanned as the disk is spun. By altering the disk rotation speed, pinhole diameter and/or pinhole spacing; the image brightness, contrast and quality can be optimized, making SDCMs highly customizable. **However, while larger pinhole diameters results in improved light transmittance through the disk it also reduces resolution, and while having big pinhole spacing eliminates any pinhole cross-talk, it also reduces light transmittance.** Having small pinholes with large spacing would result in higher resolution but the lowest transmittance of light through the disk. Transmittance can be improved with the addition of a second disk which contains micrometer-scale lenses in the place of pinholes. The micro-lens disk focuses light through each pinhole of the primary disk, with considerably improves light transmittance to the sample, as seen in figure below. | *A standard Nipkow-Petran disk used in SDCMs. Hundreds of pinholes are arranged in Archimedean spirals (left), which pass over the sample as the disk spins. The pinholes have diameter* $D$ *and separation distance* $S$, *changing these values can optimize the resultant image received.* |
> | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/What-is-spinning-disk_3.png) | *Using a secondary micro-lens disk to improve light transmittance through the primary pinhole disk. Laser illumination is focused through the pinholes by the micro-lenses, with both disks spinning in sync over the sample. Image obtained from Graf et al. 2005.* |
> | An important point to consider is that SDCMs excite/image multiple points at once and can spin at very high speeds, meaning that high-quality scientific cameras are needed in order to capture images from the sample. These cameras need to have high framerates due to the speed of the spinning disk, because longer exposure times or a slow disk rotation speed may result in a streaky image. By using cameras that can match high rotation speeds, high-quality images are produced | **SDCM offers a clear improvement on LSCM and conventional fluorescence microscopy, allowing for fast and efficient imaging of live samples, dynamic processes and optical sectioning of 3D samples in 2D slices.** Multichannel images can be taken at multiple wavelengths, resulting in high-quality images dense with information, and deconvolution methods exist to further improve contrast and image quality. **The main disadvantage of SDCM** is that only a small proportion of light is transmitted through the disk to the sample, but with customizable pinhole disks, secondary micro-lens disks and highly sensitive cameras these limitations can be easily overcome, while also improving imaging speed and field of view even further to produce high resolution images faster than ever. |

| ![whentouse1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/whentouse1.png) | ![whentouse2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Optical%20Sectioning%20and%20Confocal%20Microscopy%20(Kurt%20Thorn)/whentouse2.png) |
| -------------- |---------- |

## Two Photon Microscopy (Kurt Thorn, UCSF)
> [course site](https://www.ibiology.org/talks/two-photon-microscopy/) | [YouTube](https://www.youtube.com/watch?v=CZifB2aQDDM) | [Two-Photon Microscopy](https://www.sciencedirect.com/topics/medicine-and-dentistry/two-photon-microscopy) | [ Cambridge Advanced Imaging Centre - Two-photon imaging](https://caic.bio.cam.ac.uk/microscopes/two-photon/two-photon-imaging) | [MicroscopyU -  Multiphoton Microscopy](https:/www.microscopyu.com/techniques/multi-photon/multiphoton-microscopy) | [Helmchen, F., Denk, W. Deep tissue two-photon microscopy. Nat Methods 2, 932–940 (2005). https://doi.org/10.1038/nmeth818](https://www.nature.com/articles/nmeth818) [*Sci-Hub link*](https://sci-hub.se/https://doi.org/10.1038/nmeth818) | [Larson, A. Multiphoton microscopy. Nature Photon 5, 1 (2011). https://doi.org/10.1038/nphoton.an.2010.2](https://www.nature.com/articles/nphoton.an.2010.2) | [Two Photon microscope](https://www.youtube.com/watch?v=shl00D7bgZg) | [**`Lang: HUN`** Gergely Sz. et al - Háromdimenziós, gyors, ketfoton pasztazo eljarasok](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/H%C3%A1romdimenzi%C3%B3s%2C%20gyors%2C%20ketfoton%20pasztazo%20eljarasok.pdf) | [ WEIRD Fluorescence Two-Photon absorption!](https://www.youtube.com/watch?v=Uxc5H5Ui1n0)
>
> This talk introduces two-photon microscopy which uses intense pulsed lasers to image deep into biological samples. It can be used for imaging thick tissue specimens or even imaging inside of live animals.

One way to circumvent this problem came in 1990 with [2-photon laser scanning microscopy](https://www.photometrics.com/learn/light-sheet-microscopy/introduction-to-light-sheet-microscopy), which – back then – was meant to replace single-photon confocal microscopy (Denk et al., 1990). In principle, the use of near-infrared and infrared wavelengths could penetrate much deeper into a sample and was gentler to it (when compared with excitation using shorter wavelengths) as there was less phototoxicity. However, the cost was an almost prohibitively expensive Titanium-Sapphire laser which has only barely decreased in price today. Furthermore, as 2-photon laser scanning microscopy is a scanning technique, the acquisition is very slow – or if fast, very noisy. Large samples take hours to acquire and the only way to increase speed is to make smaller regions and restrict the field of view.

![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/confocalvstwophoton.png)

[Multiphoton microscopy](https://burke.weill.cornell.edu/imaging-core/research/research-equipment/two-photon-microscope-thorlabs) (also called non-linear laser scanning microscopy) utilizes simultaneous absorption of two or more photons for excitation of a fluorescent label. Fluorescent emission, following two-photon excitation, is exactly the same as emission generated in normal one-photon excitation. Out-of-focus light is eliminated by excitation only at the focal plane; no pinhole is necessary. In contrast to confocal microscopy, several different fluorescent labels can be excited with the same wavelength (e.g. Alexa488, Alexa 568, and Cy5 are excited by ~ 800 nm; GCamp6f, DsRed, and Evans Blue are excited by ~920 nm in living tissue). This microscopy is best suited for deep imaging in living and fixed tissue, as well as in vivo animal imaging (several mm deep from the surface of a cleared or transparent sample).

| ![tissuepenetration](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/tissuepenetration.png) | ![scattering](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/scattering.png) |
|----- | --- | 

Confocal microscopy utilizes a pinhole to exclude out-of-focus background fluorescence from detection. Thus, this technique allows three-dimensional sectioning into thicker tissues. However, the excitation light generates fluorescence, and thus produces photobleaching and phototoxicity throughout the specimen, even though signal is only collected from within the plane of focus. This large excitation volume can cause significant photobleaching and phototoxicity problems, especially in live specimens. Furthermore, the penetration depth in confocal microscopy is limited by absorption of excitation energy throughout the beam path, and by specimen scattering of both the excitation and emission photons.

| ![scattercleared](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/scattercleared.png) | ![scatteringsolved2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/scatteringsolved2.png) | ![imagingininfrared](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/imagingininfrared.png) |
|----- | --- | --- |

The phenomenon of two-photon excitation arises from the simultaneous absorption of two photons in a single quantitized event. Since the energy of a photon is inversely proportional to its wavelength, the two absorbed photons must have a wavelength about twice that required for one-photon excitation. For example, a fluorophore that normally absorbs ultraviolet light (approximately 350 nanometers wavelength) can also be excited by two photons of near-infrared light (approximately 700 nanometers wavelength) if both reach the fluorophore at the same time (see Figure 1). In this case, "the same time" means within an interval of about 10 × E(-18) seconds.

| ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/two_photon_energy.png) | ![twophotonexcitation](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/twophotonexcitation.png) |
| ---- | --- |

Because two-photon excitation depends on simultaneous absorption, the resulting fluorescence emission varies with the square of the excitation intensity. This quadratic relationship between excitation and emission gives rise to many of the significant advantages associated with two-photon excitation microscopy (discussed in more detail below). In order to produce a significant number of two-photon absorption events (in which both photons interact with the fluorophore at the same time), the photon density must be approximately one million times that required to generate the same number of one-photon absorptions. The consequence is that extremely high laser power is required to generate significant two-photon-excited fluorescence. This power level is easily achieved by focusing mode-locked (pulsed) lasers, in which the power during the peak of the pulse is high enough to generate significant two-photon excitation, while the average laser power remains fairly low. In this situation, the resulting two-photon-excited state from which emission occurs is the same singlet state that is populated when carrying out a conventional fluorescence experiment. Thus, fluorescent emission following two-photon excitation is exactly the same as emission generated in normal one-photon excitation. Figure 1 presents a Jablonski diagram illustrating absorption of a single (ultraviolet) photon (Figure 1(a)) and the simultaneous absorption of two near-infrared photons (Figure 1(b)), producing the identical excited state.

| ![noautofocuslight](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/noautofocuslight.png) | ![tissuescattering](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/tissuescattering.png) | 
| ---- | ---- |
| ![scatteringrevisted](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/scatteringrevisted.png) | ![rasterscattering](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/rasterscattering.png) |

The likelihood of a [TP excitation event](https://www.sciencedirect.com/topics/medicine-and-dentistry/two-photon-microscopy) depends on two factors:
1. spatial and temporal availability of excitation photons, and
2. TP cross section of the molecule, referring to the physical properties of the molecule favoring TP absorption.

| ![twophotonmic](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/twophotonmic.png) | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/detectorconfigoftwophotonmic.png) | 
| ---- | ---- |

In practice, two-photon excitation microscopy is made possible not only by concentrating the photons spatially (by focusing of the microscope optics), but also by concentrating them in time (by utilizing the pulses from a mode-locked laser). The combined effect allows generating the necessary photon intensities for two-photon excitation, but the pulse duty cycle (the duration of the pulse divided by the time between pulses) of 10 × E(-5) limits the average input power to less than 10 milliwatts, which is just slightly greater than that used in confocal microscopy. Although the laser pulse durations are considered ultra short, typically ranging between approximately 100 femtoseconds and 1 picosecond (10 × E(-13) to 10 × E(-12) seconds), in comparison to the fluorophore absorption event of about 10 × E(-18) second, they are relatively long in duration.

| ![lasers](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/lasers.png) | ![lasers1](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/lasers1.png) | ![lasers2](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/lasers2.png) | ![laserspectra](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/laserspectra.png) |
| ----- | ---- | ---- | --- |
| ![lasernottwice](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/lasernottwice.png) | ![multiplemeissionchanels](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/multiplemeissionchanels.png) | ![https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/fluorphore_excitation.png](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/fluorphore_excitation.png) | | |

The laser power required to excite a fluorophore-containing specimen has an optimal limiting value. With increasing laser power, the fluorescence intensity increases, up to the point of fluorophore saturation. The condition of saturation occurs at a laser power that is sufficient to cause a significant proportion of the fluorescent molecules to exist in their excited rather than ground state (this power level is about 1 milliwatt at the specimen for one-photon excitation, and 50 milliwatts at the specimen for two-photon excitation). At higher power levels, the additional photons are simply unable to excite more fluorescent molecules. Any excitation energy increase beyond the saturation point contributes to increased photodamage and photobleaching. Each experimental setup must be assessed for damage imposed during beam scanning, and it is important to recognize that trivial cell viability tests (such as esterase activity or dye exclusion) do not always accurately reflect cellular photodamage. For many experiments, more rigorous functional tests may be more informative. As examples, in one recent published investigation, the viability of hamster embryos was confirmed by their continued development, and in another the viability of pancreatic islets was confirmed by their maintenance of normal glucose-stimulated NAD(P)H response.

![whentouse](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/images_from_the_videos/Microscopy%20Two%20Photon%20Microscopy%20(Kurt%20Thorn)/whentouse.png)

## Light Sheet Sectioning (Ernst Stelzer, European Molecular Biology Laboratory)
> [Original article](https://www.ibiology.org/talks/light-sheet-sectioning/) | [YouTube](https://www.youtube.com/watch?v=lteywF6wKu8) | [MicroscopyU - Light Sheet Fluorescence Microscopy](https://www.microscopyu.com/techniques/light-sheet/light-sheet-fluorescence-microscopy) | [Wikipedia - Light sheet fluorescence microscopy](https://en.wikipedia.org/wiki/Light_sheet_fluorescence_microscopy) | [Maura F., Cory B., Jaclyn W. - Light Sheet Microscopy](https://www.edmundoptics.com/knowledge-center/application-notes/microscopy/light-sheet-fluorescence-microscopy/) | [Omar E. O., Jordi A., Emilio J. G., Pablo L. - Light-sheet microscopy: a tutorial](https://opg.optica.org/aop/fulltext.cfm?uri=aop-10-1-111&id=381035) | [Teledyne - Introduction To Light Sheet Microscopy](https://www.photometrics.com/learn/light-sheet-microscopy/introduction-to-light-sheet-microscopy) | [Your Next Confocal Should Be a Light-Sheet Microscope](https://www.bruker.com/en/products-and-solutions/fluorescence-microscopy/light-sheet-microscopes/why-choose-a-light-sheet-microscope.html) | [Photometrics - Light Sheet Microscopy](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/related_documents/Light-Sheet-Microscopy-App-Note.pdf); [*original file*](https://www.photometrics.com/wp-content/uploads/2019/10/Light-Sheet-Microscopy-App-Note.pdf) | [Rüdiger P. - Gaussian Beams, https://doi.org/10.61835/mla](https://www.rp-photonics.com/gaussian_beams.html)
>
> In this talk about light sheet sectioning, Ernst Stelzer discusses the new technique of light sheet microscopy, also known as selective plane illumination (SPIM). This uses two objectives, one to illuminate the sample and a second to image it, and allows long-term 3D imaging of thick specimens, like developing embryos, with minimal photobleaching and phototoxicity.

![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/comparison-of-confocal-microscopy-and-light-sheet-microscopy-b-bruker-jpg.png)

| Light Sheet Sectioning (LSFM) | Two-photon scanning microscopy |
| -------------------------------- | ------------ |
| [Light Sheet Sectioning](https://en.wikipedia.org/wiki/Light_sheet_fluorescence_microscopy) is a fluorescence microscopy method that has medium-high optical resolution but good optical sectioning capabilities and high speed. | Two-photon microscopy allows examination of deeper layers of a sample. |
| In this technique, the sample is illuminated perpendicularly in the plane of interest only. | In this technique, fluorescent dyes emit light only when two photons are absorbed simultaneously. |
| For illumination, a laser light sheet is used, which illuminates only the currently observed slice. | Two-photon microscopy minimises sample damage and background noise. |
| This minimizes photodamage and stress in the sample and improves the contrast of the images, similar to confocal microscopy. | It is particularly useful for examining living tissue such as nerve cells. |
| LSFM allows faster imaging than conventional point scanning methods | [Two-photon microscopy](https://www.bruker.com/en/products-and-solutions/fluorescence-microscopy/light-sheet-microscopes/why-choose-a-light-sheet-microscope.html) also allows the examination of deeper layers. |

The new player is light sheet fluorescence microscopy, an illumination approach orthogonal to the detection that allows for a single plane illumination. This was first introduced by [Richard Zsigmondy](https://en.wikipedia.org/wiki/Richard_Adolf_Zsigmondy) and [Henry Siedentopf](https://en.wikipedia.org/wiki/Henry_Siedentopf) in 1903 using a slit-based system when they were working for [Carl Zeiss](https://en.wikipedia.org/wiki/Carl_Zeiss_AG). They called it the Ultramicroscope, and regrettably, it was soon forgotten. The main aim of light sheet microscopy is imaging a large sample with short time intervals under healthier conditions over longer periods (e.g. days) than compared to conventional types of microscopy or using the single plane illumination to improve single-molecule imaging (FCS, TIRF).

The main aim of light sheet microscopy is imaging a large sample with short time intervals under healthier conditions over longer periods (e.g. days) than compared to conventional types of microscopy or using the single plane illumination to improve single-molecule imaging (FCS, TIRF).

> ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/What-Is-Light-Sheet_image1-1024x580.png)
>
> Looking closely at the illumination of a widefield microscope we notice that the sample which is placed in the plane of focus – where the energy of the excitation beam is the highest – is also excited above and below the focus ([Fig.1A](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/What-Is-Light-Sheet_image1-1024x580.png)). The sample is moved to various planes of focus to acquire a z-stack and by this, the researcher obtains 3D information about the structures of interest and relation to secondary structures. If this was performed in a live cell experiment, planes above, in and below the focus would be exposed constantly which is detrimental to the sample. Aside from photobleaching, phototoxic side effects will cause the sample to be in non-physiological and later in apoptotic conditions, resulting in at least in extensive photodamage and even cell death.
>
> One could assume that using a point scanning device such as a confocal microscope would improve the situation as only individual points are excited. This is not the case because the beam,
>
> a) needs to be focused onto a certain point and thereby will also above and below the point of focus, and
> b) as the beam resides only briefly at each individual point, a much higher laser power is required which in turn causes even more damage to the sample ([Fig. 1B/C](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/What-Is-Light-Sheet_image1-1024x580.png)).
>
> One solution is to create a beam that is parallel to the focal plane ([Fig. 1D](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/What-Is-Light-Sheet_image1-1024x580.png) ). Above and below the plane of focus there is no excitation, and hence no additional negative effects in the out-of-focus planes. Rather than having a Gaussian beam profile, the beam is shaped by a cylindrical lens creating a sheet of light at and around the point of focus. The cylindrical lens maintains the round [Gaussian beam](https://www.edmundoptics.com/knowledge-center/application-notes/lasers/gaussian-beam-propagation/)’s profile in one dimension but compresses it in the other, as seen in Figure below.
>
> ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Cylindrical-Lens.png)

The light sheet allows the system to be designed in a way that only the sample plane in the focus of the imaging lens is illuminated. The emitted light is picked up by the imaging lens and detected by a camera. In terms of imaging, this has multiple benefits. Firstly, a single plane can be acquired in one quick snap due to multiplexing with the pixel array of the camera (benefit of the widefield system), and secondly, by illuminating one plane and avoiding exciting too much the planes above and below the imaging plane, the signal-to-background is improved (benefit of confocal in the z-axis). A light sheet diagram is shown in this Figure.

![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/Light-sheet.png)

In most light sheet systems, the majority of the optical components remain static while imaging. To obtain an image in 3D, the sample needs to be moved across the light sheet (other variations using adaptive optics sweep the sheet in XYZ across the sample). This is usually achieved by the movement of the sample holder which is a 3D- (XYZ) or 4D-stage (XYZθ, where θ is rotation). The movement in XYZ allows imaging of larger samples which can be larger than the already large FOV, which will require stitching in post-processing. The rotation gives access to the sample from multiple angles which can then be fused together. Because of this fusion process, the image will gain quality because the resolution will improve (see below for details).

Apart from that, there is a massive time benefit to be gained by this approach. Scanning a comparably sized sample on a confocal over minutes to hours can be reduced on a light sheet system to seconds and minutes. As mentioned before, phototoxic side effects are highly reduced by a factor of between 20-100x. And this is the main goal of light sheet microscopy: Image under physiological conditions for much longer and obtain more relevant data in live-cell experiments. 

### Light Sheet Optics
> #### Gaussian Beams
> In many laser optics applications, the laser beam is assumed to be [Gaussian with an irradiance profile](https://www.edmundoptics.com/knowledge-center/application-notes/lasers/gaussian-beam-propagation/) that follows an ideal Gaussian distribution. All actual laser beams will have some deviation from ideal Gaussian behavior. The $M^2$ factor, also known as the [beam quality factor](https://www.edmundoptics.com/knowledge-center/application-notes/lasers/beam-quality-and-strehl-ratio/), compares the performance of a real laser beam with that of a diffraction-limited Gaussian beam. Gaussian irradiance profiles are symmetric around the center of the beam and decrease as the distance from the center of the beam perpendicular to the direction of propagation increases. This distribution is described by Equation:
>
> | $$I(r) = I_0exp( \frac{-2r^2}{\omega (z)^2}) = \frac{2P}{\pi \omega (z)^2} exp( \frac{-2r^2}{\omega (z)^2})$$ | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/gaussian-beam-propagation-fig-1.webp)  <br> *The waist of a Gaussian beam is defined as the location where the irradiance $1/e^2(13.5%)$ of its maximum value* |
> | ---- | --- |
>
> **Where**
> - $I_0$  is the peak irradiance at the center of the beam
> - $r$ is the radial distance away from the axis
> - $\omega (z)$ is the radius of the laser beam where the irradiance is $1/e^2(13.5%)$ of $I_0$
> - $z$ is the distance propagated from the plane where the wavefront is flat
> - $P$ is the total power of the beam
>
> > #### $M^2$ *(Beam Quality and Strehl Ratio)*
> > *Once the laser’s performance is known, defining the true performance of any optical system used with it will allow for an understanding of the final system performance. Comparing the real performance of an optical system to its ideal, [diffraction-limited](https://www.edmundoptics.com/knowledge-center/application-notes/imaging/limitations-on-resolution-and-contrast-the-airy-disk/) performance is done by using the Strehl ratio.*
> > $$M^2 = \frac{\pi \omega_0 \phi}{\lambda}$$
> > **Where**
> > - $\omega_0$ is the beam waist
> > - $phi$  is the divergence angle of the laser
> > - $\lambda$ is the lasing wavelength
>
> However, this irradiance profile does not stay constant as the beam propagates through space, hence the dependence of $\omega(z)$ on $z$. Due to diffraction, a Gaussian beam will converge and diverge from an area called the beam waist ( $\omega_0$ ), which is where the beam diameter reaches a minimum value.
> | The beam converges and diverges equally on both sides of the beam waist by the divergence angle $phi$ (Figure right). The beam waist and divergence angle are both measured from the axis and their relationship can be seen in equations. | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/gaussian-beam-propagation-fig-2v2.webp)   <br> *Gaussian beams are defined by their beam waist (* $\omega_0 *), Rayleigh range (* $z_R$ *), and divergence angle (* $\phi$ *)* |
> | --- | --- |
>
>  $$\omega_0 = \frac{\lambda}{\pi \phi} \Rightarrow \phi = \frac{\lambda}{\pi \omega_0}$$
> **Where:**
> - $\lambda$ is the wavelength of the laser
> - $\phi$ is a far field approximation
>
> Because of this, $\phi$ does not accurately represent the divergence of the beam near the beam waist, but it becomes more accurate as the distance away from the beam waist increases. Also as seen in the equation above, a small beam waist results in a larger divergence angle, while a large beam waist results in a smaller divergence angle (or a more collimated beam). This explains why laser beam expanders can reduce beam divergence by increasing beam diameter.
> 
> 
> The definition of Gaussian beams concerns both the intensity and phase profile, as explained in the following.
> 


The drawback of investing more light (or using short-wavelength with higher energy) is that, particularly when looking at living organisms, large amounts of phototoxic side products are created which degrade and eventually kill the organism. This makes it hard to say whether the observation is taking place at a healthy time point or at a decaying state.

> Light-Sheet Fluorescence Microscopy is [the method of choice for long-term, high interval (minutes to days) live sample imaging](https://www.bruker.com/en/products-and-solutions/fluorescence-microscopy/light-sheet-microscopes/why-choose-a-light-sheet-microscope.html). Most of the models used in confocal microscopy are suitable for light-sheet microscopy. Due to its unique capabilities, additional challenging specimens are included in the spectrum of samples that can be acquired with a light sheet microscope.
>
> Compared to confocal laser scanning and spinning disk confocal microscopy, light-sheet microscopy enables fast, high resolution, true volume, and in-depth imaging.
>
> ### 1. Low Photobleaching
> > [Photobleaching](https://www.microscopyu.com/references/fluorophore-photobleaching) (also termed fading) occurs when a fluorophore permanently loses the ability to fluoresce due to photon-induced chemical damage and covalent modification.
> 
>  Long-term exposure to light, especially in time-lapse studies, induces photobleaching, hindering the detection of the fluorescent molecules.
>
> A comparison of the photobleaching rates of light-sheet microscopy, spinning disk, and confocal microscopy reveals a reduction in photobleaching when working with light-sheet microscopy. The effect is already visible when imaging a single plane @ 100 fps, but the difference becomes particularly astonishing when comparing imaging of a stack of 40 µm (1 µm steps) @ 100 fps.

> ### 2. Low Photo-Toxicity
> > [Phototoxicity](https://pubmed.ncbi.nlm.nih.gov/28749075/) frequently occurs during live fluorescence microscopy, and its consequences are often underestimated. Damage to cellular macromolecules upon excitation light illumination can impair sample physiology, and even lead to sample death. In this review, we explain how phototoxicity influences live samples, and we highlight that, besides the obvious effects of phototoxicity, there are often subtler consequences of illumination that are imperceptible when only the morphology of samples is examined. Such less apparent manifestations of phototoxicity are equally problematic, and can change the conclusions drawn from an experiment. Thus, limiting phototoxicity is a prerequisite for obtaining reproducible quantitative data on biological processes.
>
> Light-sheet microscopy stands out for its effective use of excited photons, which minimizes phototoxic effects. This contributes to prevent the generation of misleading and artificial results. The comparison of light-sheet microscopy and spinning disk microscopy images revealed inappropriate bone development in zebrafish due to photo-damage in spinning disk microscopy.

> ### 3. High Temporal and Spatial Resolution
> Light-sheet microscopy enables high imaging speed and the possibility to capture a higher number of events. This is of particular relevance in fast occurring dynamic processes.

> ### 4. Optical Sectioning
> > The ability to image thin sections without having to mechanically slice a thick specimen is afforded by [optical sectioning microscopy](https://zeiss-campus.magnet.fsu.edu/referencelibrary/opticalsectioning.html). Sections are achieved by eliminating the excitation and detection of fluorescence that originates in regions removed from the focal plane.
>
> | ![](https://github.com/gabboraron/ibiology---Microscopy-Series/blob/main/realted_images/optical-sectioning-bruker.png) | Optical sectioning refers to the generation of clear images of specific focal planes within a 3D structure. Good Z resolution enables the 3D reconstruction of a sample. |
> | ---- | ---- |
> | Fluorescence microscopy, e.g. confocal microscopy, spinning disk confocal and light-sheet microscopy, enable optical sectioning. Confocal Microscopy and [Spinning Disk Microscopy](https://www.photometrics.com/learn/spinning-disk-confocal-microscopy/what-is-spinning-disk-confocal-microscopy) image a specific focal plane by point scanning the sample and rejecting out of focus fluorescent signal with a pinhole(s). These techniques enable high-resolution image acquisition at the expense of photo-damaging effects and/or high time consumption. | Light-Sheet Microscopy offers [intrinsic optical sectioning](https://gonzales.science/resource-blog/intrinsic-imaging) by the specific illumination of one particular focal plane. This is achieved by the orthogonal arrangement of the illumination and detection objective lenses as well as the projection of a thin light-sheet on the sample. Intrinsic optical sectioning significantly reduces photo-bleaching and phototoxic effects offers high acquisition speed and the possibility to perform long-term experiments. |

Light sheet is a rapidly growing field in imaging due to its flexibility, low cost, and power as a platform. With the low phototoxicity, ability to image large samples and acquisition of images at a high speed, light sheet is the technique of choice for a number of researchers. The many flavors of SPIM allow for a customizable platform to be built in-house, for any imaging needs.
